pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed

pkg/ccl/testccl/workload/schemachange/schemachange_test.TestWorkload [failed](https://mesolite.cluster.engflow.com/invocations/default/4290429d-c71e-4fe1-b57a-2e91b71f3c57?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL3Rlc3RjY2wvd29ya2xvYWQvc2NoZW1hY2hhbmdlOnNjaGVtYWNoYW5nZV90ZXN0) on master @ [f09d8777b3b1fbb2b9f0ed9e41ba3db36c6bad30](https://github.com/cockroachdb/cockroach/commits/f09d8777b3b1fbb2b9f0ed9e41ba3db36c6bad30):


```
    "result": true
   },
   {
    "query": "SELECT ARRAY[parent.table_schema, parent.table_name, parent.column_name, child.column_name] FROM (SELECT conname, conkey, confkey, conrelid, confrelid FROM pg_constraint WHERE (contype = 'f') AND (conrelid = 'schema_w0_26.table_w0_139'::REGCLASS::INT8)) AS con JOIN (SELECT column_name, ordinal_position, column_default FROM information_schema.columns WHERE (table_schema = 'schema_w0_26') AND (table_name = 'table_w0_139')) AS child ON conkey[1] = child.ordinal_position JOIN (SELECT pc.oid, cols.table_schema, cols.table_name, cols.column_name, cols.ordinal_position FROM pg_class AS pc JOIN pg_namespace AS pn ON pc.relnamespace = pn.oid JOIN information_schema.columns AS cols ON ((pc.relname = cols.table_name) AND (pn.nspname = cols.table_schema))) AS parent ON ((con.confkey[1] = parent.ordinal_position) AND (con.confrelid = parent.oid)) WHERE child.column_name != 'rowid'",
    "result": null
   }
  ],
  "previousStatements": [
   "INSERT INTO schema_w0_26.table_w0_139 (\"col̩139/_w0_140\", \"co\tl139_w0_141\", \"co\rl139_w0_142\") VALUES ('D4F8C15A/45877B8C':::PG_LSN,2950:::OID,'[6.37572e+06,-1.2034172e-12,1.4185538e-34,-1.1256203e+35,1.861421e-22,2.9406678e-16,2.9659254e-11,0.0037588933,5.440584e+16,-1.1130043e+20,1.2724256e+23,9.250852e+31,6.188661e-26,6.466282e+27,1.9801252e-31,2.2391767e-14,2.4566156e+27,-1.5270432e+06,2.0606345e+09,1.2932242e-15,-1.5464317e-37,-1.0718978e-11,1.0515704e+19,3.281661e+37,-873886.5,0.00011976923,-1.2514879e-24,8.0830436e+15,9.3692553e-35,6.743873e+11,4.301777e-09,3.4767015,1.2548509e+06,-3.6112828e-11,1.4136248e-38,1.4633678e-20,4.0169663e-22,4.1057243e-05,4.7946162e+31,-3.956189e-27,2.8255177e-38,9.226787e+33,-1.5831998e+19,-1.7536478e-26,-8.704113e-26,1.570242e+22,1.4436456e-37,-1.5911105e+10,-1.6163829e+28,-1.0778505e+37,-1.1658975e+23,1.0770524e+34,1.2854016e+27,-7.392551e-18,5.670549e-18,330384.62,-27.053762,-2.2514658e+08,2.8875034e-29,-1.5116403e-12,-1.5478444e-06,-1.2083175e+20,-0.00013328542,13179.89,4.614189e+32,9213.93,2.1994207e+21,1.2223597e+25,-1.5534452e-23,-148440.98,-7.2073905e+06,-1.2885851e+31,3.943444e-12,3.558375e-39,-1.3034836e-14,2.935731e-13,-5513.9316,-4.2058084e+35,38762.973,2.0084888e+09,4.834162e-15,-1.2934668e+28,-1.6066451e-25,-4.1429578e-35,2.0105442e+16,-5.13892e-27,56.175556,-3.576617e+09,-3.371862e-29,-4.669953e+36,2.4197607e+11,-1.03417934e+15,-2.7350162e+23,9.7777524e+33,5.1105792e+08,1.666455e+28,-1.2012179e-08,-1.0882481e+11,0.015887091,3.692576e+28,1.5399109e-13,-5.8204986e+23,7.3241786e-29,1.8415353e-25,-12535.724,-209940.42,-2.6109966e+19,3822.1987,-3.2940367e+25,-6.059258e+17,-4.2437268e+07,-2.137088e+13,-1.1946817e+37,-2.2938832e-36,-2.4400962e+38,3.4642642e+31,1.5493944e-05,3.3819065e+19,-6.065148e-34,1.8876291e-30,-7.245447e+25,-72.4522,-1.4227209e+29,-1.8741007e+15,-1.639019e+38,3.9839848e+23,4.3685052e-35,-2.6044793e-25,-6.4550093e-21,1.430345e-21,-5.1819833e+33,-1.5856982e+09,8.7256916e-32,1.7401065e+37,-6.2617305e+06,-24.60156,-2.4565308e+11,-256780.84,2.387655e+21,-9.395601e-17,-5.1845274e-14,3.5341359e+15,1.766633e+21,-2.6332655e+37,7.005046e-36,-3.1109955,1.2157421e-05,-11301.48,0.0017230993,21508.64,-1.25213455e-05,4.739158,2.353178,0.0027734712,-2.1371208e+17,-1.4179565e+20,2.31053e-30,-1.78573e-23,3.0154338,2.0896041e+09,9.149717e+11,-5.771317e+31,5.918492e-15,1.13599234e-13,-1.7241773e-33,6.2662502e-37,6.152844e+34,7.8626746e+28,-7.111786e-12,59.554085,7.369288e-34,-1.4467889e+19,-1.4098465e-10,9.1184716e-24,3.9164504e+29,2.1228325e-20,9.139491e-17,281230.84,-3.4318737e+17,9.953243e+20,-0.00055754144,5.813484e+06,-6.4441786e+31,-2.7950332e-15,-3.2001238e-12,-0.00018160482,1.981376e+20,-1.8229726e-34,-2.7455273e-22,8.083466e+14,-1.01011705e-26,0.010575661,1.6136928e+34,3.3959583e-22,-7.263346e-31,1.3320828e+13,-23759.404,-6.3066394e+11,-4.4545323e+37,-4.197033e-14,-0.004756049,0.006559397,-1.1011522e+28,-5.129003e+18,-7.3289375e+23,8.896424e+27,2.3058813e+29,1.8573404e+20,-0.0020665168,-6.5700546e-38,-3.455685e-16,4.1464366e-23,6.217583e+23,2.3228047e+19,-8.3110266e+36,9.902227e-34,-1.8417702e-16,7.627211e+37,10671.517,-7.19717e-40,-0.011192191,-1.0065183e+23,-8.692975e-37,3.089334e+29,-1.5394236e-30,0.13694897,-0.0006001079,2.052704e+33,-1.130317e+08,-5.5190344e-30,3.6230857e-28,-2.5631437e-20,-5.1917768e-18,-6.3927056e-35,-3.4966364e+25,-5.9244158e+29,2.2644025e-25,-4.37587,-6.3520685e+16,2.7126103e+37,2.1298327e+20,9.327025e+20,6.026364e+19,-7.3737056e-06,4.0626486e-28,2.4539882e-21,3.599991e+13,2.688586,1.0405534e+32,-1.4667593e-33,-5.7813184e-05,-5.2934544e+22,-5.006397e-38,1.0052091e-24,-1.9211349e-16,-1.8124908e-06,2.6920107e-24,-1.9847818e-11,-3.0280988e-25,3.5325776e+19,1.6052428e+32,1.6039048e-18,0.003788979,-3.9670827,-0.02657735,-1.8659823e+24,10091.644,-1.7192843e+14,5.6592617e-06,3.1503492e+30,3.5957806e-10,1.4463776e+18,84.679886,4.4002435e-10,-9.981939,2.0596959e+30,-1.0653941e-22,-3.143746e+23,2.2942896e-23,2.9417908e+29,3.524879e+09,-5.207586e+37,-3.110625e-10,-2.1434279e+12,3.2888245e-33,2.416562e-20,-3.684229e+25,-2.6002841e+36,2.1895963e-28,-747.5083,-0.017061034,-6.9958583e+22,1.2475765e+30,-1.3606612e+21,3.3707546e-18,-4.8024693e+23,-5.351087e+09,1.3869181e-05,7.6086894e-16,-4.7439077e+23,-1.6845871e-32,1.4363604e+34,-0.23070703,1.04929205e-23,-1.02771394e-19,-2.7122402e+17,5.291101e-19,-1.1475983e-13,45493.527,8.6142064e+30,1704.0302,-2.7206604e+07,5.8100663e-16,1.0889025e+31,3.730004e-21,1.3699083e+21,-7.0850774e-21,-3.4785773e-19,1.6374112e-21,3.3242604e+18,-8.236281e-31,-0.00016780339,-1.8722818e-34,4.374808e+09,-2.5315394e+07,-1.0142632e-24,-32.857864,4.9779468e-20,-6.225776e-19,-2.6745468e-14,4.3802526e-29,-8.4448075e-08,-7.444393e-10,2.100677e-15,-0.02560837,2.5489878e-26,-5.3834605e+13,-8.430297e-06,-3.9822242,-1.5979018e+19,5.890916e+31,-3.779074e+16,5.9874323e+08,1.262315e-29,-7.201657e+32,-1.7320313e+15,-2.7997165e+31,6.6699514,-6.889446e+16,-1.106155e-29,5.45947e-23,1.0305697e-24,-1.3113076e-33,4.6635844e+07,-1.1571168e-18,-4.16258e-26,-3.6956948e-07,-9.427946e-39,-7.5063375e-37,-2.13686e+31,-4.1761326e-38,-1.0773168e-36,-1.8320048e-25,6.2530764e-38,-3.0174832e+24,-4026.513,-2.582029e+38,3.9807163e-22,-0.056840632,-1.2795652e-16,46.711464,4.597291e-31,-0.0005997498,1.0306507e+22,-1.1925454e+30,-3.9504168e-38,5.6002186e+26,5.737558e+21,1.8612206e+09,7.3453807e-29,3.5097182e+23,4.762181e-39,-7.8917965e-09,9.111711e+16,-7.1245456e-35,-2.553571e+26,-3.5578387e+15,-1.0518353e-11,7.027007e-24,2.7851702e+09,1.8212596e+27,6.353963e-22,3.5983192e+10,2.307626e-23,3.783367e-10,-21218.86,-6.1372898e-33,3.2331268e+20,6.400674e-05,1.2299001e-29,6.446349e-34,6.683316e-23,5.0119323e-19,-1.5035643e+36,-2.9141286e-15,-1.03186566e-22,1.1805628e-32,1.0727496e+20,-3.5035285e+26,4.0160185e-08,-2.3064896e+26,-3.1492166e-16,-5.4026698e+23,-124.99262,3.0561905e+16,-7.8415793e-28,-2.5064126e-15,9.690715e+10,9.786526e-06,-6.519533e-34,9.2467334e-10,-5.826023e-09,-1.5675213e-35,-22529.617,2.7974093e+32,-2.934276e+33,-3.1795557e+23,-4.1796486e+24,2042.6469,1.3433207e-18,20.444635,2384.8525,-3.5745173e-32,-9.353501e+08,100028.51,-7.3311347e-35,3.6726992e+24,3.4420612e+25,-1.8398655e-22,2.4694374e+21,-7.0843093e+16,-1.5059459e+10,-6.2709606e+30,-1.602121e-06,1.1746724e+21,-29886.275,-173836.2,-2.3856567e-22,-7.5123423e+15,-8.062432e+21,-2.0872231e-07,0.0001055678,-2.259025e-39,-199515.77,0.10839179,34.174744,9.495057e+28,-7.376465e-07,-3.0890035e+06,-643.9723,1.05461536e+09,-1.9447658e-25,9.211441e+08,5.625846e-26,-6.8814505e-33,4.7236376e-19,3.9705086e-31,-1.9841609e+11,333704.4,-1.8638897e-33,-4.410783e+10,2.3968522e+16,4.092269e-37,-2.622214e-36,57.594563,-1.9727806e+07,0.04569412,-111647.41,-251930.23,-5.3882895e+32,-7.9642483e+21,3.0957996e-31,-1.7418804e+36,2.1294167e+25,-1.1762607e-12,1.6300138e-23,9.553642e-24,-3.3850509e-31,3.2928017e-13,7.723677e-20,-8.373517e-14,1.1634747e-24,-5.8032005e+09,-1.1858047e+26,215.80351,8.3185265e-26,861.39465,6.902003e+15,1.8448397e-37,9.50533e-07,19706.072,1.570806e-12,-2.3091716e+14,-5.8945658e-21,1.889287e+33,-1.9338477e+19,1.5259438e-05,-1.6735016e-22,-3.0958032e+23,2.6633419e+17,-0.0069454205,-1.0000436e+07,1.0138134e-12,6.182693e-11,-1.9154098e-07,1.4243284e-27,-6.9453e-06,2.513933e-26,3.2009562e+34,-8.317185e-05,-3.111561e+27,0.0039512785,7.907909e+29,1.0758086e-14,2.2136843e-09,3.8778312e+37,-1.5278002e-18,515142.5,-3.6074483e-13,-4.4784906e+17,1.2664793e-18,-0.021790972,-1.7921184e+25,4.1151405e-17,-6.6747904e+15,-1.2243573e-34,-5.4510427e-13,-1.2392896e-22,7.821041e-06,2.8186348e+20,7.129557e+26,6.1335865e-27,3.9644647e-19,3.2010584e-07,1.8876553e+29,-2.5656527e+17,-4.682957e+10,0.014439766,-5.9188793e-08,9.182874e+23,-1.13243726e+11,1.3191059e-08,6.745244e+07,5.681462e+11,3.953307e+15,4.0634733e+21,-1.4724887e-16,-6.517555e-28,-3.0011024e+18,14645.023,-4.3601545e+14,4.0321507e+24,-8.072015e-13,6.437476e+29,-1.2905467e+31,-5.1221985e+19,2.80872e+38,-5.8146825e+23,3.662109e+30,-1.6053437e-26,-3.8594513e+28,-5.219565e+34,-2.9404688e-32,-1.037488e+08,-1.6956847e-31,4.3657312e+12,-0.057618253,-1.09645046e+15,6.6325035e+19,-9.385938e-10,5.338179e-37,6.1717104e+31,6.9523075e-11,-1.6054115e-11,2.1874838e+38,96.98373,-1.3525248e+32,-2.7597119e+37,-2.2315355e-07,2.045259e-37,1.3723573e+20,2.8448704e+34,3.5440264e-37,-1.221247e+27,-1.2349147e-27,-7.9146835e+34,-8.469857e+25,-3.308289e-08,-2.8440805e+25,-1.0806483e+26,-27.672264,1.3349216e+18,8.361852e-08,-8.2386347e+37,1.3848334e+17,4.818682e-11,-3.8029707e-06,8.057087e-10,-2.464961e+36,-4.2465796e+22,-2.5463562e+36,0.0012133927,8.62292e+12,6.93178,-80.35212,1.329126e-16,6.684256e+21,-5.827006e+22,8.284183e-22,-2.403869e-11,-1.5193483e-16,-3.8551112e+18,11.898447,-2.5018318e-23,2.241859e-06,-6.957504e-13,5.0878745e-20,5.8897056e+18,-7.4889726e+27,2.17548e-24,-1.03205095e+18,-3.322192e+35,-4.4113896e+12,-7.5662103e-07,-1.0626786e+14,1.280606e+07,-1.5394132e-25,-9573.58,5.4120497e-21,3.9513833e-22,2.6942518e+12,3.728511e-09,8.403999e-05,212.68169,-5.845698e-36,-1.28695825e-23,-3.0075097e+34,-1.5071609e-34,-1.01465145e+23,1.3317023e+28,1.2722867e-07,0.042672705,1.1584487e-30,5.718752e+07,1.4977548e+34,1.4874687e-20,-1.1313392e+38,-6.3331224e-14,1.138761e-07,-1.3455732e-22,3.732154e+35,3.4269455e-14,-6.5099345e+35,3.796388e-18,-1.2099831e+37,7.0203865e-33,4.9897192e+19,-2.324364e-16,2.8001481e-31,-2.2959408e+27,-5.701386e+06,-1.6801038e+11,-3.043784e+14,-7.652292e+11,1.2093914e-30,3.3276298e+08,-1.905939e+36,6.335071e+09,-5080.217,-1.0838328e+10,6.195102e+16,1.9595074e-08,-1.1454988e-37,1.4954113e-27,-5.5716718e-27,1.4264693e+37,877619.9,-8.643164e-07,6.803001e-16,-2.256283e+25,4.6650045e-19,-53034.082,-4.361054e-36,-3.2844788e-37,-8.634223e-33,6.8837375e+26,0.31455633,-208.36809,6.849062e+22,-2.4244358e+23,-7.0987163e-23,3.2640173e-25,3.6098369e+18,-1.0636303e-09,6.017374e+28,3.4881635e+30,-1.1002822e+19,3.0145154e+18,1.4131205e+08,1.0437436e-09,1.252973e+06,-1.1835518e+20,1.1736163e+08,-6.2097446e-05,-8.621787e-22,-190254.02,-2.6873037e-08,6.5030015e+20,1.1313383e+07,-57685.598,1.1578009e-29,1.6998867e-28,394.39914,0.020710815,31556.402,8.240663e+25,-2.0390207e-32,2.2264355e-11,9.17176e-23,2.228495e+28,3.949602e-22,-2.2120073e+10,2.7331578e+16,-1.1571996e+27,-4.677521e-08,7011.5005,4.5993105e-26,1.7939815e+15,6.7629197e-12,-1.01259655e-32,-1.640347e-21,1.1158554e-32,7.195293e+16,-1.6448177e-23,-0.08385823,-6.32551e-15,2.280145e-11,-2.8996093e-22,4.8669326e-11,0.0016605151,-1.2532695e+08,-1.1382809e-33,9.123268e-12,-1.2119818e-13,87438.82,51.94331,3.777535e-19,-9.149008e+26,6.341936e+15,4.0983698e+12,1.1599403e+25,6.7540244e-32,2.749205e-11,233.26007,5.3881952e+14,1.758113e-20,0.14961757,1.691749e-05,0.079684034,1.6646533e+28,-1.949874e+21]':::VECTOR),('D75E60C1/E1E7BEED':::PG_LSN,1042:::OID,'[8.9555014e-32,9.573613e-36,-3.513527e-38,2.784832e-36,-4.912747e-15,-1.420616e-30,-5.8223003e+24,-1.6952909e+38,1.3017803e-34,3.2146507e+18,-13.634786,7.665985e-09,2.2569727e-23,9.328549e+31,-1.840939e+14,-882670.4,-0.016830415,3.818172e+27,-8.9390405e+32,-0.085320614,-6.356373e+13,1.186961e-39,-1.3932572e-17,-1.3040148e+32,-0.0014080838,-1.0741286e-27,-3.1362863e+19,6.003816e-26,2.2445731e+32,-4.039437e-33,-1.3005172e-37,-1.01126774e-16,-9.381563e+10,-1.8930403e+08,-2.1005716e+09,-4.2520946e-18,-1.7103982e+16,-1.08330165e-32,231.5382,1.5107122e-09,0.00017229255,-0.009933038,2.323389e-31,1.2378626e+10,2.228844e+34,-7.15613e+07,9.2520915e-30,0.5438685,-5.6711993e+23,6.596475e-26,1.6969062e-19,5.9170015e-22,1.1951764e-38,1.2137404e+09,6.4466576e-07,0.0007441927,-3.4037049e-37,-4.700727e-28,-3.5449197e-37,2.3815692e-35,-2.9749275e+10,3.1022722e+21,-6.080302e-13,-2.4272423e-10,-4.4843265e+12,1.1311057e-33,2.8946793e-11,-2.5742977e+28,-3.8594104e-07,7.2073817e+31,-7.557432e+25,2.650635e+37,4.9640185e-35,225.71832,-8.723731e+24,-1.6759639e-05,1.50938385e+23,1.5931879e+22,-809.0182,1.8561293e-31,-7.1332855e-18,6.8569578e+25,9.077371e-23,3.8617036e-30,1.1574508e-29,2.124288e+21,2.8801204e+20,-3.285233e-07,4.3996525e-07,-4.903289e-13,1.4348443e+29,-6.96719e+10,1.5167475e-09,-6.4557856e-24,-1.0607309e-26,-3.6536986e-16,4.3110626e-11,-0.00028720096,6.2710983e+29,-2.8466307e+23,-1.2536993e+30,7.3248627e+27,-1.53163e-39,-1.1545265e+21,4.345337e+12,-3721.536,-3.469334e-29,-5.919198,-6.7429686e+12,2.0826456e-05,-3.1883003e+28,0.003396788,-1.2876124e-24,-5.7451597e-19,-1.3913444e+29,1.17197834e-35,6.974376e+13,2.175113e-22,1.2691095e-21,9.682493e-38,-2.8687564e-38,-4.8439634e+12,-1.8870051e+24,2.0785297e-26,3.7140345e-19,-0.0030716043,1.9534488,2.0329389e+09,4.228595e-33,-2.5017058e-10,1.6085478e+23,-2.4579917e+35,0.03393853,-5.251587e+37,2.8852904e+34,-9.784016e-13,-5.106286e-34,2.687286e+10,-1.2531071,4.709755e+26,-9.248315e-17,197032.34,1.370891e-31,-7.351256e+12,1.1697114e+16,5.91171e+26,-7.8740645e+15,3.3842973e-09,-4.6267295e-16,7.239556e+08,2.633549e+32,-1.9070544e+07,7.480536e+17,-2.5935651e-23,-3.4420882e+06,-2.5549803e-17,2.726065e-08,-1.4478959e-09,-0.00084853225,-1.4781918e-34,9.950831e+20,1.2073915e-18,-177.24388,-6.88507e-38,-9.802967e-28,-6.9191665e-06,60429.656,-7.650032e+24,-2.8690805e-22,3.5350292e-32,6490.4795,-1.20525345e-20,-420.3813,-5.8740643e-22,1.9361936e-30,1.990076e-28,1.369695e-11,-8.9595746e-13,2.5124237e-35,-5.4996325e-26,1.6137913e+14,-0.0044617965,-1.16318485e-20,-8.1433026e-19,6.7696005e+06,1.5605574e+34,2.2910425e-36,1.4459683e-30,1.7716432e-37,3.1762152,-8.530503e-13,4.0449832e-13,-8.477118e+22,-6.934759e-23,2.1729202e-08,2.9316182e-22,2.2774744e+13,-3.6856758e-09,2.6225389e+38,-9.404819e-24,-44.75461,0.24179521,-6.7093695e-09,-9.579244e-07,931.3213,-3.1387258e-22,-771912.94,4.435467e-37,-243354.11,3.5184237e+22,-0.085384704,-1.2564468e+11,-2.1020044e-08,-6.8406176e+24,-0.28227943,4.618929e+17,-7.999238e-32,-8.32827e-08,-7.5369725e-35,-1.22605915e+17,83465.84,5.6935175e+06,6.772036e-07,-4.122949e-16,1.0304032e-08,-2.5298378e-27,-2.4170831e+36,-0.004855795,7.2836413e+19,3.690615e-06,3.7922857e+12,-3.2974043e+12,1.7676649e-16,-3.2900841e-15,1.9566043e+26,36999.863,-1.0672601e+08,-2.603806e-14,3.2178806e+08,1.7506466e-26,6.098086e+18,-2.1089577e-33,-2.2036517e+08,5.18622e+31,-1.700341e+16,1.451308e+22,1.3102875e-08,1.3706951e+34,-4.8156104e+31,-9.275482e-16,-2.7614842e-08,-2.4314437e+09,1.672659e+07,5.374623e+09,-4.4562123e+31,9.5530745e+30,5.1798228e+22,-1.57768e-23,6.902859e+21,-863469.3,2.2279758e-35,-1.0840931e-35,-1.9527701e+32,9.819708e+06,1.1148426e+29,2.5425018e+06,8.162147e-24,-1.9649066e-07,-4.854977e+25,-6.2524078e-24,8.401423e-32,-7.918775e+29,9.943542e-25,-2.5127284e+12,-3.723766e-19,4.188579e+16,-3.7673143e+24,1.6338413e+17,-25075.18,6.751609e+24,-49643.17,2224.363,-1.5852048e-19,1.772114e-09,-9.525537e+23,-1.1128464e+22,8.5912536e-05,2.8253848e-16,2.8858497e+19,2.2099338e-32,-3.066337e-37,-5.805256e-39,-6.706897e+18,-4.2813763e+21,-2.1229406e+11,-7.903807e+26,3.8644506e+37,-8.507789e-32,-4.110523e+27,-2.1110111e+21,-2.5200117e-23,-1.8053515e-05,1.3077514e+13,-0.92257947,1.8882824e+27,-3.4073812e-23,-137.06494,-2.4479702e-08,-2.0900505e-10,-4.255088e+13,-3.7697702e+31,-4.3662816e+28,3.8875295e+06,2.549666e+11,521103.88,1.2407591e-05,1.4782618e+29,2.6822886e-18,-1.3911157e+22,-8.380427e+18,9.2106444e-14,1.7762517e-13,8.5719013e-13,2.3565198e+22,6.262121e+32,3.7211707e-37,-1.6909004e-36,2.0665845e+36,1.8907996e+23,1.3064808e+16,4.432193e-10,-1.1134414e+35,-7.7588195e+22,6.6701824e-34,-8.058061e+37,-2.7493009e-28,9.321167e-32]':::VECTOR),('FFFFFFFF/FFFFFFFF':::PG_LSN,1042:::OID,'[-2.9937195e+36,-3.7322695e+13,-1.1255593e+31,-1.7526052e+14,-2.368396e-31,-2.8696793e+32,-1.05121036e-32,1832.3954,-5.178436e-08,-5329.877,-2.7507204e+26,2.4106452e-09,6.472493e-20,7.5692615e+06,-2.094807e+15,4.1886915e-30,1.0055974e-08,1.0690882e-22,1.4661364e+27,1.9992185e+24,-3.0923207e+35,9.324534e-39,-0.00037688788,2.8122514e+09,-1.5043412e+28,-1.7267448e+20,-106.49737,-85.81815,-1.5092318e-13,6.653923e-08,1.5466738e+06,7.211758e-16,-8.243447e+33,-3.9760357e+17,4.889579e-27,3.7508857e-11,-1.1496678e-22,-6.2554894e-27,-3.6314592e+08,6.0300373e+32,-1.3448893e+24,-85535.08,7.9968035e+27,8.2627463e-38,-2.0566054e-31,2.2351615e-29,-8.370359e+20,1.023916e-38,-0.0040051052,9.05531e+15,-5.2587522e-14,2.0968821e-29,-2.1806073e-07,-9.615397e+06,3.0478354e-22,-1.128984e-25,1.5945786e-36,1.32410176e-17,-3.92261e+11,-3.1912418e-21,-7.232008e-11,2.403707e+36,8.658483e-20,-5.798588e-06,-7.0924914e+37,0.38469863,-6.037151e-11,-1.7644013e-23,7376.273,-5.0487857,-2.4754715e+14,-0.002262757,-2.4177228e+22,-4.7059984e+17,6.0646265e-21,9.5861644e-05,82173.17,3.1686955e+19,1.0606352e+38,28967.264,-4.08161e-36,-1.3517449e-31,4.6994318e-38,3.7244506e+23,-1.7364046e-10,1.716024e-25,-1.1355675e-06,5.557291e-24,1.5424417e+11,-0.61957425,4.8005014e+23,-3.5631716e-30,-1.33906e+30,-7.6435906e-22,7.474537e-14,3.7132993e+18,7.595627e-10,-5.6845465e-14,-9.938369e-37,-5.8961764e+19,2.178732e+06,-4.755604e-08,-1.3015316e+29,2.713169e+28,3810.431,0.0095234355,6.37502,1.3289193e+33,3.756978e-23,-1.2118323e-31,-0.00029119014,9.978305e-15,7.61723e+30,-1.7177775e+15,-181.72568,-3.1392144e+15,-5.495939e-32,21.835306,-6.1723566e+29,6.4672597e-37,-4.4858057e-06,-8.1054015e-35,1.8592942e+08,-2.6371708e-30,-1.3307758e+32,2.2328092e-36,2.0355172e-08,-0.001803364,-6141.899,-9.462225e-25,1.8722972e-17,1.9588027e+34,-2.0803793e+15,-1.3657275e+14,4.0447217e+09,5.19572,-7.3734384e-37,-0.013250299,1.2910163e+36,-3.9929466e+23,-1.26800265e-36,0.00015285912,-2.1766166e-35,-7.6144064e-35,-5.247701e+13,-1.2813683e-36,-1.3907345e+23,-5.776904e-15,-2.2251086e+29,29.440683,-1.38250435e-27,-9.2636566e+36,6.756121e+22,-1.0444154e+12,-5.1592364e-30,1.4927781e+28,-1.7673173e-36,9.95059e+09,2.7401478e-17]':::VECTOR)"
  ],
  "usesDeclarativeSchemaChanger": true
 }
}
    schema_change_external_test.go:134: 
        	Error Trace:	pkg/ccl/testccl/workload/schemachange/schemachange_test_test/pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:134
        	Error:      	Received unexpected error:
        	            	***UNEXPECTED ERROR; Received an unexpected execution error.: ERROR: duplicate key value violates unique constraint "table_w0_139_col_w0_139_w0_217_key" (SQLSTATE 23505)
        	            	(1) forced error mark
        	            	  | "fatal error when running txn"
        	            	  | github.com/cockroachdb/errors/withstack/*withstack.withStack::
        	            	Wraps: (2)
        	            	Wraps: (3) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*opStmt).executeStmt
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/operation_generator.go:3180
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).runInTxn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/schemachange.go:499
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).run
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/schemachange.go:587
        	            	  | pkg/ccl/testccl/workload/schemachange/schemachange_test_test.TestWorkload.TestWorkload.func4.func6
        	            	  | 	pkg/ccl/testccl/workload/schemachange/schemachange_test_test/pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:123
        	            	  | golang.org/x/sync/errgroup.(*Group).Go.func1
        	            	  | 	golang.org/x/sync/errgroup/external/org_golang_x_sync/errgroup/errgroup.go:78
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1695
        	            	Wraps: (4) ***UNEXPECTED ERROR; Received an unexpected execution error.
        	            	Wraps: (5) ERROR: duplicate key value violates unique constraint "table_w0_139_col_w0_139_w0_217_key" (SQLSTATE 23505)
        	            	Error types: (1) *markers.withMark (2) *schemachange.ErrorState (3) *withstack.withStack (4) *errutil.withPrefix (5) *pgconn.PgError
        	Test:       	TestWorkload
Schema Workload Stats
Total Schema Statements Executed = 112
Total Schema Statements Succeeded = 112
Total Schema Statement Expected Failures = 0
Total Transactions Committed = 76
Total Transactions Rolled Back = 50
Total Transactions Executed = 126
    schema_change_external_test.go:99: backup, tracing data, and system table dumps in /var/lib/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/testlogs/pkg/ccl/testccl/workload/schemachange/schemachange_test/test.outputs/logTestWorkload3479606553
    panic.go:626: -- test log scope end --
test logs left over in: outputs.zip/logTestWorkload3479606553
--- FAIL: TestWorkload (31.49s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestWorkload.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>


Jira issue: CRDB-42294

Epic CRDB-40419

pkg/ccl/testccl/workload/schemachange/schemachange_test.TestWorkload [failed](https://tanzanite.cluster.engflow.com/invocations/default/09d589ce-cd76-429d-99af-be5c713ed93c?testReportRun=8&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL3Rlc3RjY2wvd29ya2xvYWQvc2NoZW1hY2hhbmdlOnNjaGVtYWNoYW5nZV90ZXN0) on master @ [83589fb87caa92fb42e83994f1691978f37e4cbb](https://github.com/cockroachdb/cockroach/commits/83589fb87caa92fb42e83994f1691978f37e4cbb):


```
{
 "workerId": 1,
 "clientTimestamp": "05:51:44.191281",
 "ops": [
  "BEGIN",
  {
   "sql": "INSERT INTO public.table_w0_12 (\"col12%v_w0_13\", col12_w0_14, \"c%qol12_w0_15\", \"c%pOl12_w0_16\", col12_w0_17, \"c̘%pol\"\"12_w0_18\", c͊ol12_w0_19, col12_w0_20, col12_w0_21) VALUES ((-96):::INT8,'[3.6895382e-31,1.7597513,-1.4390969e+12,9.984367e-31,6.2392696e-20,-3.6250834e+35,4.7244834e-15,-12.8477745,-2.2037097e-15,-5.1286876e+09,4.0789795e-32,-5.2723392e+36,0.06425001,-2.853838e-38,-1.9862132e-26,-7.968525e-38,3.449915e+26,-3.5385511e-31,-5.985287e+07,30.42268,-2019.7423,-1.6385513e+22,-1.9990332e-31,-3.393465e-27,-1.9870603e+34,975.78766,0.00015912655,1.1406458e-15,-1.2141469e-05,-1.3191997e-23,-1103.8004,-1.7382156e-11,-1.5243706e+15,1.3819344e+14,-1.6870325e-29,2.0326092e-26,-5.9317024e-16,-2.7005498e-26,-3.658212e+09,-6.65481e+13,1.8933331e+37,-2.0055107e+31,6.123789e+28,1.9957512e+25,-0.0057965927,5.900979e+23,-3.2666163e+19,-7.4127555e-34,-39.90166,6.845762e+37,0.0005261615,1.1188177e-19,0.9243889,-5.906735e-08,1.60485e-34,5.298848e+11,5.6032844e+13,-9.460735e-36,9.154857e-19,-7.560572e-12,-1.724302e-38,5.6174904e-22,28.137253,2.6268931e-14,1.3907224e+11,39.113888,9.6780115e+23,-9.838324e-26,8.60525,-6.1819585e+27,-3.341537e-28,6.18101e+06,-5.977831e-11,6.7219445e+37,3.6389579e+37,3.1596211e-31,-2.4090697e+23,1.2141997,-3.7366946e+23,-4.586212e+09,-2.3707824e+08,1.1172854e+20,5.7882684e+29,-8.2110285e-32,-5.9642214e+08,-7.9609253e-22,-1.0467689e+15,5.587411e+25,-4.678952e+07,-5.979355e+20,8.9369205e-06,-6.4802234e+32,-4.526017e+34,9.966559e+27,1.597469e-11,2.5828128e+08,1.2039148e+16,7.220302e+27,-2.4893457e-10,4.317606e+11,2.9489982e-05,-6.51299e-05,-3.6913856e-13,-831879.4,-1.3539407e-08,0.4001762,2.5572233e-37,-4.9429433e-11,6.1200165e+15,-0.011735535,8.238468e+11,-1.7484187e+13,-1.3315395e-16,-1.060367e-11,1.6876215e+31,6.8409626e-08,2.851793e-29,-1.9031675e+33,3.8599365e+31,-3.4057242e+26,1.8586203e+17,-1.7278324e+33,-8.803253e-31,2663.3447,2.9806764e+33,-53.228855,-1.9841562e+32,-3.0716004e+18,2.799817e-23,3.6274405e+16,-10.728801,-1.3000115e+08,3.7269656e+10,2.821239,0.39952153,-3.349391e+11,5.059256e-39,7.203695e-27,-1.1691765e+22,7.218619e+32,-1.1842239e-14,1.7370163e-21,6.328988e+33,4.1105763e+14,-1.4139309e-37,4.6484954e-27,1.4958884e+36,-5.674957e+28,-5.4655493e-07,2.1284552e+29,27621.621,-9.911958e+20,2.1469093e-07,-2.2755118e-27,2.18546e-34,-3.5958174e+37,1.0941391e-08,3.815271e-16,1.8968472e-24,-2.7828216e+23,3.738776e-22,-1.815295e+06,3.2540992e+31,-1.0353506e+14,-4.0739482e+09,-1.1382495e-25,-1.1640036e-11,4.7192874e+26,-7.19295e-24,3.332089e+24,-9.3065064e+07,-7.123941e-16,-5.1939797e-29,-2.5325229e-23,-21420.492,1.1886609e+32,-0.32359925,-1.2963304e-07,-4.3798785e+06,4.54947e-14,-0.00058371894,9.41724e+19,1.5598914e+07,-2.318031e-17,-2.3265418e-08,2.8869477e+35,-2.8731111e-33,-1.3331029e+21,-2.3675013e+17,1.8827612e+07,8.624211e-29,-137965.16,-4.3427136e-08,0.0044445386,6.423506e+27,-1.6426964e+22,-2.6843642e+37,6.8127176e+20,-3.9443744e-36,6.1966075e-19,-4.5648837e+18,3.2739086e-30,-8.1796574e-26,-4.6939793e-33,-3.6714141e+31,-1.1875137e-37,-8.839783e-39,5.327555e+37,1.4126151e-23,-2.2857457e+14,4.8920428e-09,0.000118456526,-2.3190398e+35,3.0814906e-12,-2.1120137e+35,1.2414338e-35,-4.5556783e-07,-1.9600125e-23,-1.807185e+23,-3.7644884e+31,-8.2460736e-13,-1.9767648e-09,-5.8107624e+19,-8.257478e-33,-1.1385841e+31,5.52314e-30,2.0849909e-07,7.851517e+12,2.9587386e+10,2.6100268e+36,-5.2121644e+17,9.122929e+34,5.8621823e-31,6.896503e-12,-1.2778721e-28,-1.476752e-15,-1.455074e+38,-1.2330805e+17,6.8923144e-17,-1.822304e+38,2.5784753e+19,2.0123664e-13,2.6312627e+36,9.790179e+12,2.499826e+35,7.3349032,-3.0650863e-05,-0.0016479988,-9.89388e+31,1.1062483e+09,249.86795,8.767872e-21,-1.2419963e-29,2.9518579e+23,150.70639,-1.4385328e+38,-1.0862413e-32,-1.9981504e+26,-3.4635093e-09,7.4009795e+27,5.53741e+19,-2.5734033e-33,4.38648e+37,1.0582469e+08,-9.975329e+14,0.0043696514,6.220293e+30,-2.484342e-07,6.1020115e+29,7.750774e-21,7.6079014e-28,-7.5198614e-08,8.865778e+10,6.71965e-24,7.186287e+07,1.8487877e+19,-3.5790097e-19,-1.0583698e-38,-1.1329636e-12,6.9916237e-09,7.3796565e-30,-3.2480437e-32,-3928.066,-1.03905663e+12,7.090958e+08,-9.049452e-28,-2.8936975e+16,7.517554e-20,-1.1105939e-13,9.77727e+20,-2.0193268e-06,3.87462e-05,5.4076863e+14,5.798818e+13,-1.0833998e-31,9.935226e-29,-3.1808411e+34,-1.1558709e-34,-8.944671e-26,5.4249836e+09,2.1177907e+23,-176258.98,2.9745656e+23,-977178.6,527.9674,3.9925416e-32,-1.3458567e+16,2730.857,357990.06,-9.810146e-36,-2.1137422e+14,2.7978872e+15,-5.4893556e+30,6.0987155e+31,-9.4062885e+16,-34500.855,-2.0150241e+36,2.0497595e+21,-1.1804685e+07,-0.003489317,-3.4504612e+24,2.9446351e+26,3.6485786e-23,-18.345387,2.870783e+14,-3.5446085e+31,-9.174715e+34,1.2575765e+25,3.7302525e+34,2.2443098e+14,-1.8332161e-06,6.261767e-08,-3.1101914e-23]':::VECTOR,'\"c\"':::JSONB,'00:55:18.207367-07:54':::TIMETZ,false,'''CLj'':151A,328C,541C,685C ''EWoxQmLXT'':261C,517A,679A,755C,951A ''Gszjhps'' ''gaS'' ''nptVlyJD'' ''zs'':32C':::TSVECTOR,'[3.737965e-23,-4.1904043e+11,4.9510505e+11,-6.5324867e-25,-1.4899887e+09,-3.0187814e-25,-1.845662e+08,1.35916665e-08,0.029117536,3.4543754e-31,3.2273953e+34,2.3138553e-31,-2.3987951e+32,-1.4396635e-21,-6.000827e-26,2.3402937e+16,-3.5806996e-25,-5.825019e+25,1.26177416e-26,7.0452487e-13,-9.963991e+22,3.2834963e+15,-2.9551063e-16,3.4578457e-18,-2.939244e-20,5.451675e-22,7.3955005e-35,1.7616252e+20,8.4349684e-32,3.1227475e+37,-3.5845348e+26,-5.4811856e-32,-2.3829427e-06,4.30261e+15,4.9991086e+19,-0.00055504177,-4.7884743e-27,1.04836755e+21,1.2561566e+31,-6.920776e-29,-5.406975e-38,6.9267417e-19,0.015442196,2.6920103e-35,-1.3824951e-19,-3.8152632e-07,4.4054516e+13,-1.3177415e+11,42676.562,2.2589583e-18,1.5692952e-21,-1.3199185e+32,-5.4533952e+17,-5.796172e+18,-4.4048034e-21,-1.4042435e-10,-8.859304e-09,475.13342,4.711197e+31,-7.032565e-36,-5.5246423e+26,-1.5026343e+27,-2.5796154e-09,-1.9212738e+24,-5.7364425e+06,2.3029022e+22,-1.0274151e-32,-3.3250092e+18,1.0379543e-29,5.9558627e-31,6.571073e-25,-6.5954324e-18,1.5981018e-32,-1.9655342e-35,-9.4638645e+27,2.1499856e+07,1.1090099e+13,-1.4017517e-22,-154.89432,3.6271405e+12,-1.2323153e-17,-4.0572696e+30,4.1667954e+12,-4.3045748e-05,-4.046995e-35,-5.091896,-2.0096098e-37,7.670645e+29,2.6034092e+12,7.471466e+33,-4.753158e-35,-1.12921885e+21,-3.5154393e-22,3.3283152e-23,4.6612822e-17,-9.185313e-21,-1.211149e+11,3.2917056e-33,-6.448225e+10,-26156.574,5.009118e+28,1.2034972e+38,2.0899518e+34,1.5180865e-28,-9.3575013e-26,-2.5594075e-16,-1.0313965e-33,0.0045289705,-8.801765e+22,-1.29628435e-11,-3.3430292e-27,4.244925e+24,-9.197319e-07,-6754.795,-2.2058556e-08,3.8955582e+23,-4.1789207e+22,1.8903582e-06,1.1632769e+34,-2.0191272e-05,2.99459e-07,-1.9875318e-14,6.627963e+30,4.263955e+24,-1.0923827e+34,8.776001e-06,-4.7958597e-32,1.8024196e-15,1.6585675e+18,6.8751274e-28,-2.282874e+24,1.6111889e+24,-1.14184084e-20,2.0408145e+17,-2.5778465e+10,-254310.06,-1.0249071e-24,-2.0328319e-38,-1.38952545e-14,2.2807494e-19,-2.7231398e+24,-6.84852e-24,-1.7322309e-24,3.0467833e+36,4.1138796e+36,-7.611199e+15,-5.3266067e-17,1.114008e-20,4.4741797e-11,392965.78,-4.5429928e-24,2.662961e+19,-6.594115e-11,9.37152e+30,-3.5219576e+23,-2.8143684e+20,-2.4125221e-19,7.5850065e+19,-1.6905275e-30,0.018082945,0.00053906045,1.6527666e-16,-9.4615256e-26,5.484057e+35,1789.8002,-4.6598618e-17,6.6642385e+06,7.3759473e+22,9.7904814e-30,-5.1283916e-19,-5.739345e+19,969238.4,4.260851e+24,8.476119e+22,-4.3361624e-29,-2.073352e-25,-7.784068e-29,3.842737,-8.713333e+15,-8.0045803e+36,4.945184e-26,-7.794079e-24,-31.627127,-9.603177e+11,7.0550597e-13,-9.968123e-22,-4.239188e-22,-2.2807586e-17,-3.5561578e-37,7.462421e+13,-64.931725,-8.0367057e+27,-3.622066e+23,-5.7095685e-17,1.5326296e+38,7.141774e+33,-4.6758515e-23,3.3938136e+34,1.5758936e+36,1.0047274e-37,1.0623714e-05,308.565,-3.3976966e+37,3.7092276e-18,1.0870151e-23,-1.4215504e+34,-1.1820188e+28,4.8999746e+24,-1.276854e-28,-5.4032606e-24,-5.597219e-34,-1.2550759e+16,-3.741664e-36,-2.532089e+12,-3.8677257e-22,-4.8174698e+08,-6.598857e+27,1.5445832e-22,8.299526e+20,3.4982868e+30,5.905083e-18,-5.4355583e-06,2.3938559e+33,7.623543e+06,-0.00024958185,1.2979742e+10,-2.2602733e-16,-8.42667e+08,5.229392e-24,5.9767714e+20,-9.9638276e+30,7.0856554e+31,-1.0377397e+31,-7.7146863e+15,-1.7673242e-25,-8.051726e+18,3.6555132e+24,3.5356984e-12,1.3689132e+26,-31311.617,-1.8853646e+14,1.908004e-18,5.0932128e-21,-2.079827e+29,2.8696773e+28,-3.6353e-34,-5.404666e+11,1.9957035e+19,-9.861995e-17,-4.6520583e+20,9.003119e-17,1.8247782e-05,-7.82077e+17,-2.6556527e-37,2.1826994e-23,-5.649529e-05,9.260498e+33,-9.252404e-32,3.3150954e+33,3.7839866e-35,-5.0010524e+07,-0.24266128,-2.2331214e+10,-1.3376057e+16,6.709785e+35,-781.3645,1.303878,-8.799335e+15,-2.3588279e-23,3.2831911e+16,-5.4639732e-34,7.314309e-39,-1.8708203e-19,-1.21445514e+30,0.0144202225,12525.231,1.05485355e-19,6.7966706e+25,-3.1325949e+32,-2.0981828e-05,36.49979,-0.49762312,1.2863595e-07,-5.74252e-20,475.93246,-1.9765385e-07,6.7821647e+11,0.0627688,-3.752937e+25,-1.5020453e+10,-1.4347064e-27,-7.531552e+07,-3.329477e+15,1.6734361e-08,-8.0160865e-26,-3.0048308e-29,-4.148854e+30,-0.046356272,1.9595747e-37,3.905487e-32,1.3585081e-09,2.3694796e-37,1.9863133e+34,1.217165e+09,0.21088819,-2.6266598e+12,-2.9488289e+28,1.5502931e-17,2.3516686,-4.2958507e+20,-24.968897,4.40982e+19,7.5421576e-30,3.7281463e-35,-1.656824e-07,-6.0819318e-24,-4.5778616e+07,2.6322258e+35,1.2540138e+22,-2.013185e-23,1.0652864e+32,2.934879e+33,-4.21705e+29,1.2449983e+16,-3.4170083e-31,1.277348e+22,2.6581513e-18,-1.9005138e-32,6.4758624e+13,-0.00024041056,2.0307744e+38,-5.4704667e+31,-1.2522398e-10,2.7998917e-23,-3.4414174e+29,7.261414e+36,4.780923e+29,-2.0064364e+35,-3.0982616e+15,-2.6199074e-38,-1.0498818e+28,4.5586843e+30,-3.0300882e-19,3.6737212e-08,6.467603e+15,-2.5849462e+22,-0.051216193,2.0681028e+24,1.7546648e+25,-3.1700421e+22,0.05279606,-0.00011643761,-713019.6,-0.20132078,9.2719475e+26,2.9640436e+10,-3.6242173e+25,0.019749193,-1.0215071e+11,-5.65644e-20,2.3489695e-22,0.00018746342,4.7572123e+17,1.6929514e-16,-7.372345e+36,1.5542245e-38,-2.8583382e+25,-1.7362426e+21,-7.702149e+10,-4.0139193e-36,1.6309021e-37,-4.5012817e+17,2.7247504e+21,8.232758e+32,1.6809648e+31,5.933187e-23,1.6992705e-16,-1.0048039e-13,-2.7772174e+18,2.510206e+06,1.16113364e+36,9.955467e-06,-1.7764435e-34,-7.879806e+25,1.602958e-06,486.81528,-5.193107e+07,0.11956576,2.9351446e-25,3.8088464e+28,3.8433042e+30,-9.608341e+21,8.8891484e-11,-4.1616852e-11,4.32694e-15,1.1702978e-37,-2.8772795e+18,4.576232e-13,6.709147e-06,-6.057002e+14,0.12669854,-8.826521e-22,-3.642799e+16,1.4059407e+33,-3.9419603e+11,-2.2376476e-09,1.0605378e-18,974.7445,8.497272e+31,-3.0364132e+35,-1.3750297e-06,2.6138987e+08,5.781342e+34,1.697594,-3.6344262e-25,5.2278785e+14,-3.8794458e-19,-1.9827996e+15,1.981461e-36,370693.1,-1.7244684e-13,-5.461872e-21,1.5166837e-27,-2.7697463e-19,3.43706e+23,2.2348208e-11,-4.9893866e+16,4.510778e-10,-1.005555e+22,1.1245224e+38,3.522034e-21,-7.7069195e-18,1.362607e+37,3.274234e-17,1.7267574e-19,1.467011e+23,-1.0432033e+33,4.7397613e-28,-4.9558204e-13,-3.2139524e+28,-6.2423363e-31,-8.712312e+27,-8.103193e-05,-5.920147e-36,-3.5830337e+16,3.080589e-23]':::VECTOR,'010300000001000000090000001A7AFC194DDFECC191FF99D1E2BAF7C1583B534DA5A1ED4110B01446F39200C2C49248289A93FD41396C04719581FBC1005A9220736D89C15C4CD7FEE2CEE141701B2874247ACBC134F0B36FAA73E941A3A572A0953CF0C1B240AA10E6AEF64144FBAEE9611BFDC178CB193E4B8ED94102D603187DCCF3C1A09C634FAFDABFC11A7AFC194DDFECC191FF99D1E2BAF7C1':::GEOMETRY,(-7535553.126825447290):::DECIMAL),((-78):::INT8,'[-1.734002e-30,0.09103978,-1.5190519e+09,3.6237863e+15,-2.7247052e+38,9.049953e+10,9.653468e-21,-2.3513715e-29,-4.7895063e-11,-1.7117222e+20,2.3993843e-25,9.951659e+30,-4.9710202e+30,-0.0032197381,2.7624247e+22,-4.985723e-25,-6.9885604e-28,0.11856513,21848.555,7.447542e-22,2.0310463e-22,1.3314615e+20,-3.80834e-06,-2.2319912e-36,3.6976424e+36,-2.6502735e-07,-2.9639568e+07,-5.352269e+23,-9.991101e+11,4.340319e+14,-1.106563,-1.101514e+21,5.872288e-08,5.850843e-36,-1.7553042e+29,-9.163599e+18,-2.385997e-22,2.7762708e+19,-516025,4.2350098e+30,3.4435787e+21,-113152.99,9.1267665e-08,8.578123e-22,0.0064586676,-9.1427855e-14,-4.4547643e-17,1.1855393e-10,-1.9382426e-16,9.721195e-12,-9.9395114e-18,7234.5156,-102.99231,2.1683724e-11,2.64418e-17,2.4264524e+33,2.1716357e+28,-3.4902326e+13,-5.0302657e-05,4.859068e+16,-8.327397e-26,1.23706814e+33,-2.8046794e-14,2.9752946e+23,10330.766,2.3158822e-05,-7.912862e-37,-4.2194056e-26,-1.3922135e-09,9.95822e-09,2.1759042e+38,0.19687529,-1.541694e-10,-1.4016312e+16,3.3899494e-06,-4.2082948e+32,1.364347e-31,-5.7016875e+28,-7.2389088e-37,3.762731e-21,3.3087677e+36,3.832544e-06,-7.7484266e-31,-378308.1,6.984135e-28,15501.8955,-7.443164e-31,-1879.351,-0.00023294908,-2.933037e+13,-2.6390205e+25,0.00013340283,0.021688351,-7.0288795e-38,9.251658e+15,-1.5365774e-06,-1.630701e-09,262342.1,1.6081178e-13,-45828.277,-6.3310824e-12,1.2404228e-06,1.4135351e-33,1.2645901e-30,0.0068687587,-1.8006743e+32,5.9550733e-35,-440391.28,-9.170317e+13,2.0747151e+10,1.29113e+31,-6.7057165e-21,4.5099678e-18,4.988991e-18,-5.9783634e-10,6.523005e-33,-1.4919356e-05,-1.7133834e+20,-2.8436119e-30,-4.2858332e-30,1.5558305e+32,4.4045345e-23,-4.4314763e+24,-6.741743e+34,1.0116023e+18,-1.42152085e+20,2.7779538e-30,-7.5237564e-38,2.3012697e+27,-3.207402e-34,1.470842e+07,1.8954873e-34,-51862.094,5.917183e-32,-0.123619236,-6.0203603e+35,-2.092147e+26,-1.1427951e-11,1.215787e-14,-1.4742891e+33,2.5407992e+35,1.6031109e-24,-1.2269006e+38,1.4513698e+12,-6.773693e+34,5.990182e+24,-1.662132e-20,3.1539743e-18,9.168134e+21,-4.705638e-30,-1.4337842e+29,2.7948058e-35,230.39153,2.5037114e-08,-7.977378e-13,-1.0439538e-23,9.403178e-17,-4.260753e+24,-6.3923136e+16,4.5645625e+13,1.2375546e+20,-2.2405019e-17,1.26732015e+33,1.1305892e-07,-8.164988e+33,-1.7853426e-27,3.9467553e+18,-2.6348748e-36,-1.8552816e-08,7.376288e+34,-0.0013631582,-1.0185379e+16,2.32447e-40,-5.2246846e-10,-1.5139515e-28,-3.555101e-21,2.5455449e+12,3.9124217e+36,-62602.203,1.2521884e-26,-3.5467652e+20,1.1656682e+36,-1.7463914e+29,2.2974463e+16,-2.56041e+21,-4.3729563e+33,-7.7413816e+09,1.8433158e-22,1.6739173e+19,1.140482e-29,-1.08189464e+08,93.88591,-7.9793955e-07,3.1038802e-26,-1.0400654e+24,1.2780706e+07,-9.7205614e-08,4.0665524e+37,-3.243498e-38,1.2013112e+15,-3.015546e-20,3.7242388e+27,-1.5413304e-23,1.730807e-29,-0.06484917,6.725382e-33,-5.4618626e+29,3.0826537e+26,-1.0290859e-26,-2.2202973e-25,3.200475e-29,2.6114835e-15,-1.3685588e-23,-6.670452e-35,-0.00039655672,-7.234199e+37,5.433605e-26,-1.2255165e-23,-4.4773777e-37,1.6902171e-37,117516.484,-6.941082e+08,-1.7424837e-06,-5.354429e-34,-3.7954283e+28,3.3021178e+25,-9.5684634e+26,-5.5315867e-18,-5.290408e-24,3.751468e+33,-1.3975787e-31,-2.0235639e-05,8.515273e-08,-1.8295314e+28,-456664.47,-1.0579178e+12,1.6762162e+23,3.3150878e-24,223.31433,-4.692816e+09,-42346.29,411298.78,5.104061e-26,2.9161756e-05,-301.7104,-15694.478,-1.8916297e+23,3.9828293e-23,-2.8888825e+06,-5.1514442e+23,7.853796e-27,2.9489224,6.1872774e-36,0.21978064,5.83178e-07,1.1772415e+35,-1.1416933e-14,2.5426091e-17,-1.2185575e+20,4.472094e-21,-5.3316703e+35,-4.6389352e+30,-2791.5317,-15825.366,2.824183e-07,1.8292468e-37,-3.179316e-11,-2.1824559e-05,-5.6857015e-15,-4.910531e-08,8.423948e-22,1.01474866e-32,9.484968e-25,1.3526742e+31,6.50735e+33,1.6571055e-11,-4.806878e-23,-6.857437e-06,-1.0351497e+29,5.126241e+17,2.2168674e+12,-1.4096082e+37,4.1115336e+12,-1.7340116e-27,3.550055e-22,-7.7778715e+30,-1.498395e+12,-6.154821e+07,3.2719434e+21,-3.354008e+17,-5.8266583e-25,6.1469007e+19,8.927642e-36,3.727599e-38,-2.77726e+25,-4.788046e-32,2.276661e+13,-2.4052654e-12,7.541817e-10,0.0061703804,4.2741565e+22,-3.045821e+24,3.9025612e+13,1.42771e+31,3.8369884e+31,-8.6654716e+18,-1.9971031e+27,-7.7077793e+31,3.110321e-07,1.2851829e-18,-7.264523e-22,-4.139758e-21,8.158019e+21,-3.3064225e+14,-5.159458e-20,-6.922358e-28,1.0045723e+27,-4.959844e-28,6.2907694e-26,1.8605495e+06,-2.0694067e-12,9.570891e-23,-5.654761e+19,0.0048567057,-4.762526e-14,4.3825876e-16,-3.6721536e-11,-9.698629e-09,-1.7389402e+12,-3.093846e+16,8.0777256e+31,-1.5616734e-30,2.0769712e+22,2.006265e-28,-6.4042816e+33,-8.735064e+18,-1.9708579,1.6662183e+18,2.4824481e-23,-4.95427e-36,-6.244086e+08,1.4408661e+08,1.7349803e-12,-3.7394792e+06,0.015329044,-1.6443959e-14,3.3067462e-24,-115.74472,1.9395218e+23,-9.3524326e+08,-1.5943439e+14,-5.891021e+07,-1.4737435e+17,1.1053989e+17,-3.0211863e+20,7.0651443e+31,0.00022672606,7.682162e-38,-2.8359895e-05,-2.6193891e-27,8.520058e+26,-291053.6,-6.959368e-19,1.2880625e+25,-6.6520754e-38,-1839.5334,8.094083e+36,-4.200777e-34,8.51583e+25,125682.75,-1.3054835e-05,2.5179993e-22,-0.074264586,-1.2978146e+17,4.1716887e+27,-1.13261625e+20,-1.1863677e+14,8.45229e-13,1.0295522e-38,-9.8935836e-08,-8.9040126e+11,-3.7911738e-31,6.4402032e+13,-5.735561e-23,-7.6519166e-11,2.7724883e-15,4.1152654e-29,-1.0878041e-36,-5.975711e-20,1.016601e+19,8.9135336e+24,-1.5269527e+29,-5.488315e+18,8.076695e-15,-8.6688825e+11,-3.9906182e+33,0.006832255,-2.438665e-25,-1.1495006e-37,6.2496643e-31,6.569089e+34,-3.0798873e-34,4.019406e+08,-9.23962e+23,6.3509755e+32,-5.6878285e-08,-1.7683973e-08,1.0860362e-11,9.662977e+26,2.5798702e+17,-1.2356621e-25,-0.17353739,-5.3555652e+07,1.1516583e+11,-1.3372154e-28,-3.7092544e+21,0.000121298304,3.734669e-09,-3.8434198e+28,0.044375397,-12.914403,-1.6373134e-35,-1.3971629e-25,1.3698274e-06,-1.8438185e+29,-2.6137169e-11,-6.4494644e-17,2.0276983e+13,-1.5815266e+26,1.854847e+12,-7.943091e-17,-3.6979555e-20,9.392442e+33,5.4454325e+22,2.853925e+14,2.6677764e-05,1.2660803e+32,1.3793095e-19,-1.8383785e+22,-23.042576,3.5369802e+28,9.9837894e+30,-4.6442136e+07,1.9397313e-13,-2.9646128e+32,1.1981203e-05,-385521.22,-1.6247872e+31,-3.7876405e-05,2.6143184e+28,1.9156577e+28,-2.0408508e+19,6.6014826e-28,-6.865259e+22,0.00023497641,-1.5402631e-09,2.1432024e+09,1.3751888e-29,2.9827313e-30,5.3004642e-14,-0.053064644,-0.91634375,5.249107e-16,8.887847e-30,-4.386194,-5.4960964e+07,-1.4666648e-18,-2.709869e+29,1.5413132e-18,-6.407893e-32,-3.216392e+34,1.2868872e+11,1.2691105e+14,5.617968e+33,1.6447889e-15,-3.5047808e+28,4.1121787e-11,5.2730934e-17,1.1254879e+18,7.407349e+12,4.6943086e+36,1.2414106e+34,-7.176084e-37,-211.11642,-5.4765923e-32,-5.9095152e-21,2.161193e-09,-3.814339e+11,-3.3956413e+19,4.697477e-15,1208.0571,4.542404e-39,-2.7017268e-05,-4.2939987e-24,6.8044866e+15,2.535413e+31,8.922004e+26,-580682.75,2.1044975e-35,5.892428e-30,-8.53074e+25,-1.3660723e-15,-5.3516588e+17,-1.3792347e+18,-7.058381e+07,1.56630875e-27,1.7946748e+09,-2.3955954e-21,-1.00109165e-19,2.5656255e-11,1.0673592e-35,2.943261e-32,2.886465e-26,1.0377137e+12,1.4286067e+09,1.6414111e-08,8.738684e-05,-2.6614918e+06,540968,1.876985e+34,3.0330049e-30,1.5093877e-25,1.1670639e-20,4.816623e-10,2.4614857e+32,5.1234645e-31,-2.1829852e-05,1688.4143,-1.5680405e+38,2.4411631e+14,-2.7793333e+29,-1.6423374e-12,6.801326e-27,3.9037848e-33,0.00193279,1.1781498e-10,-4.2328247e-05,-1.3822789e+36,1.6969646e-09,-1.5691577e+24,2.946921e+06,7.4897428e+31,-6.5283645e-13,-3.0229379e+29,4.285583e+33,-3.334704,-4.6601358e+14,4.358054e-05,-1.756574e+06,-3.9982315e-29,-245.216,-7.2592507e-38,-9.628821e+14,-2.7189818e-36,2.9543498e+33,-5.289551e-23,3.97191e-29,1.9126689e+33,1.1698639e+06,-1.5074148e-34,-9.338134e+36,-1.4359623e+37,8.960967e+13,6.852642e-25,-3.047429e-12,-1.1916987e+15,5.0454127e-24,269.7138,3.116641e+22,-7.5669254e-07,3.2437904e-25,2.2036234e+38,-1.84038e+25,-7.4142106e+26,1.8968101e-18,-27799.963,1.3312766e+27,0.0027517225,6.344351e-18,4.1113165e-23,186.96196,5.617304e-29,877462.25,138644.75,5.980125e-09,-1.2248856e-32,-1.6179473e-11,-4.9254382e-11,1.4365156e-37,-6.291009e+08,-4.8452497e-30,-738841.2,5.5616885e+06,0.017767552,-5.63648e+17,7.4898752e+25,6.519027e+17,0.0050872536,-7.550238e-38,-1.3541629e+21,-8.554705e+32,2.3707629e-26,4.568517e-22,-2.699535e-18,-5.1711246e+19,3.1657419e+10,-3.4485102e+18,2.60069e-28,2.4854098e+36,-9.882985e-14,3.8504616e-08,-1.5401279e-10,5.826456e+21,-2.534661e-20,-0.027247569,-8.362836e+33,6.3880233e-31,-2.8039313e-10,1.3535385e+13,-3.2765736e-17,2.780248e-26,-1.9180574e+31,-2.0236573e+24,0.028548155,6.1825134e-28,1.13868e+23,-3.589324e+29,-9.749745e+27,-3.759361e+08,-2.2083999e-20,4.3481858e+23,4.2895996e+26,1.7913974e-19,1.26270475e+20,2.00968e+07,-6.604924e-07,-1.8252515e+06,2.0179419e+27,6.513831e+23,2.954859e-35,-5.0547513e-15,-7.98869e-39,-1.4668935e-30,3.5705936e-19,-2.1200503e-17,-5.2742054e-19,-1.6458048e+37,1.0819174e-27,-1.2904696e+13,-8.220336e-17,-1.9813142e+21,2.8359166e+34,-0.00061446504,-1.0491352e+35,-6.8240266e+19,1.2193811e+30,9.243519e+12,1.8305268e-07,-4.370537e+35,1.8912467e-30,5.1792557e-07,6.4687535e-19,1.0703425e+07,7.305081e+31,-1.4131208e-20,4.1218784e+36,1.7363172e+18,-9.499283e-38,-9.1107136e+08,1.5337126e-35,-4.2215593e-18,1.223644e-13,-0.00047259245,-1.19200584e+08,2.450416e-16,1.4225848e-16,-1.1553987e-30,-9.497617e+24,6.2082904e-18,2.0195443e+34,2.0626798e+07,1.4816341e-23,-1.0800434e+17,241264.61,-6.4732885e-34,1.746614e-20,-1.8445666e-12,1.4399377e+07,9.838566e+33,4.8583213e+36,1.1804938e-37,5.0597703e+35,7.403667e+16,3.028901e+28,5.6449337e-12,3.005867e-10,-1.011466e+38,4.3481798e-32,-3.7544513e-22,-7.147812e-22,-1.9814998e+09,2.6639783e+23,8.59975e-17,-9.22245e-23,6.820836e+19,5535.3853,-1.2283834e-09,3.5722303e-26,-1.831036e+26,-1.3806551e-32,5.7264984e+13,-0.9966809,-1.8770202e+23,5.2738637e+25,1.3503854e+08,2.0928705e-24,-1.1208218e+24,-4.2030612e+11,-3.5417402e-13,-9.278768e-24,-1.1933774e+37,-1.37448e+37,4.5038484e+12,4.7236754e-12,-4.429278e+08,2.306614e-08,2.2225294e-30,4.4762458e-10,6.268973e-21,-3.7629463e-29,-1.7824564e-25,2792.3396,1.9549796e-32,1.6740238e-11,-7.0053534e+13,-4.3509892e+07,-9.958582e-33,-1.272782e-21,-7.653858e+16,-2.8940754e+09,-1.2048997e-07,-6.6725466e+28,-4.703846e+07,3.4495083e+28,9.892598e+32,3.894242e+35,-3.385402e+24,-3.4155826e-25,-144.16246,-9.922266e-05,3.098939e-32,2.4302432e+24,-3.2930236e-33,6.978034e-35,-0.22387464,785606.25,-3.6445568e+06,-5.615856e+33,-210106.72,4.1243707e-37,-7.819544e-23,1.5696324e+20,9.980377e+16,7.8911165e-30,-1.1104902e-30,2.8202978e+26,-2.6312352e+38,3.2315427e-15,-9.241523e-33,-17.799095,8.8702527e+21,2.8539839e+20,-7.58184e-06,-1.402033e+18,6.2408486e+36,-3.9921993e-17,-1.0650628e+27,-9.961767e+29,5.583547e-22,-2.7154934e+32,2.2646516e-23,-1.727693e+27,-4.721931e-29,254.68393,-4.564051e-14,6.131002e-26,5.1974675e-10,-2.8363137e-23,1.6743407e+22,-4.5664686e+14,5.8599813e-21,-2.531044e+21,1.7550229e+20,-6.4517442e+22,-1.0586534e-32,-8.27195e+31,3.6754796e-37,-2.6577949e+34,9.332811e+36,-1.2544578e+15,477360.22,8.045972e+06,-3.6596958e-38,-9.932658e-18,5.6642555e+06,-1.299354e+27,-47.30457,-1.0914738e+27,-7.683521e-22,2.5127861e+36,-1.09121445e+20,-3.3529323e-36,-2.619225e+11,-5.9836207e-33,-4.407275e+37,2.146119e+35,-4.7722845e+13,3.9703197e+08,3.6058754e+21,2.8663003e-27,-7016.846,-1.4019788e+06,-4.9952214e+14,3.0235217e+11,7.1489954e-12,4.8170673e-18,-2.5344865e+38,-8.711448e+19,1.5845659e-06,1.318495e-13,-7.0064043e-26,7.2829483e+31,-1.9960774e+21,-2.1292622e+09,-4.0729214e+19,-31224.557,-3.7392856e+20,4.090224e-25,-7.378464e+06,-4.2649645e+08,1.1828651e+16,1.4901441e+09,-1.1131304e-09,4.212405e-07,1.6295999e-05,-1.367867e+38,5.839173e-10,6.228765e-38,1.6646355e-33,2.522187e-07,7.25029e-31,-4.7174398e+33,1.0889013e+18,-5.9019615e-34,6.230604e-06,-457634.12,-1.5759221e+21,7.882907e-15,-2.5792264e-13,-2.545277e+30,9.40418e+28,-5.226056e+23,3953.5637,1.14486365e-10,-2.2874056e-18,1.7396127e-09,-2.656468e-28,-1.7947793e-37,-2.6978227e+25,0.00025294247,-1.0769205e-32,-5.587031e-14,3.3962309e-09,7.6615803e-16,-0.00041573198,9.5408796e+11,-287303.12,8.057757e-32,-2.0077759e-12,2.484695e-08,-1.3832786e-23,-2.702379e-29,5.0069452e-37,1.15062625e+20,-1.7645279e+31,7.8045814e+09,4.1588115e-27,1.804043e-39,-5.894631e-37,1.1909525e-18,1.5082266e-34,-1.5708874e+30,-1.602559e-32,175643.14,41.934113,3.6068358e-27,4.1722176e+09,2.8574502e+36,323.3478,1.4312295e+35,-2.6824312e+09,-3.5329777e-18,-2.6068806e+10,7.3647348e-34,-6.944499e-06,-0.40142336,3.6292655e-14,-2.3017607e-14,5.59945e+15,7.125549e+16,0.0014960583,-998507.06,1.1815174e-21,9.834815e-13,2.7796476e-19,2.4173984e-34,2.8886333e+34,-1.2292239e-21,-1.1693334e-16,17.441092,6.5187155e-06,-1.9679326e+34,-72.947365,2.1097157e+23,1.3839e-26,-5.5295286,-8.492213e-12,-7.3438523e+37,1.3594558e+16,-2.2409849e-14,-3.0008957,1.284769e-33,-3.2141332e+36,-1.2672608,1.674372e-37,4.2966237e-37,-1.2127845e-30,1.5300232e+13,1.4050902e+20,-2.6314199e-30,4.0597538e+21,-0.017069787,-1.00272436e-10,-4.2323862e-34,2.4481318e-23,6.948916e+10,-8.433625e+28,-2.9921318e+06,-2.2642494e-28,0.0034926236,0.0034243395,-5.6360694e-09,-2.2504366e+24,-2.4314387e-08,2.063252e-26,1.608605e+22,3.0286839e+35,-8.185844e-31,-1.4582322,-3.2636875e-09,6.215736e+37,-0.00054390164,1.8048232e-30,-92009.586,239321.23,2.86243e-29,-9.7374303e-26,-3.177207e-09,-254.13017,-4.9541044e-34,2.586228e-17,8.61071e-35,3.646175e-25,4.9940937e-21,3.4765697,1.0866651e+15,0.012889616,-3.088546e-08]':::VECTOR,'[false, \"baz\", null, \"baz\"]':::JSONB,NULL,true,'''InzWVgi'' ''ZJQethGZA'':35C,267C,512B':::TSVECTOR,'[-1.3127883e+12,6.862721e+31,-5.370772e+22,-1.09005075e-17,4.4907914e+31,71525.02,2.864062e+28,-5.8402492e-37,-5.6797665e+21,-1.477171e+10,1.11213535e-13,-9.095819e-14,-1.6697568e-15,-1.6747762e-23,1.7917642e-17,-0.0005089534,1.2120322e-29,-1.3931176e-11,-7.942658e-34,4.791611e+16,1.5662592e+29,2.8572633e+13,-4.490359e-30,2.3855002e+09,-1.306024e+20,-1.0324632e+25,3.7772215e+31,-2.2861789e+23,52248.453,-1.9190461e-19,2.2159734e-19,4.6586093e-32,-1.4903796e-23,-5.322195e-07,-3.4191905e+09,-5.2162175e-23,3.6283388e+27,-3.1422876e+35,-4.928714e+12,375.53812,-7.8578234e+11,-2.0519316e+35,-45.83824,3.0720025e-27,-2.324078e-09,6.227306e-23,-6.856209,-4.1496632e+15,1.1092258e-06,2.209936e-38,4.6186505e+06,7.295095e-31,-13.48394,1.6881177e-35,4.4643445e+31,1.3945971e+38,1.826092e-06,-1.6377386e+31,-5.126654e+27,2.3930483e-30,-6.5363235e+14,1.15281826e-32,-5172.766,5.0492713e+36,8.821288e+30,1.08853864e+08,-1.5049858e+38,4.4426615e-10,-6.7869084e-17,8.752785e+18,-3.7865446e-33,-1.6732516e-12,-0.08126643,-3.1626354e-27,-9.78355e+31,1.2462002e+23,2.8367997e+24,7.1083073e+27,-4.1458666e-34,2.5536367e+12,-0.0054263403,1.9749485e+06,-2.4104198e-32,2.6016949e+08,1.6002746e+21,0.0008576455,-7.419001e+11,2.3175285e+37,-1.9808645e-35,2.0938314e-35,2.7355552e+31,-1.1142368e-15,5.0402457e+23,0.15784293,-1.29443815e+29,-2.8409126e+19,-7.0639746e+28,-5.853876e+29,-31482.27,-2.0873662e+14,-1.10046024e+30,130.47232,-1.4419585e-12,5.716411e+26,-5.309769,0.0012651843,-5.4199646e+22,7.188386e+25,2.6503538e+30,-2.7806437e+31,-3.7074825e-16,-7.9772115e+10,-1.8310178e-13,-5.814239e+06,-1.3584173e-33,-6.0914754e+21,-1.4467761e+22,-7.138752e+36,9.260012e-14,1.091965e+12,-5.6422115e-36,0.0001550063,8.032651e+33,-32032.686,-3.741081e-11,4.3593256e-29,-1.7288742e+35,2.3037196e+26,-3.5707667e-30,-1.9762669e+28,-3.178791e-34,-2.1866742e-20,4.4299842e+15,-1.7916034,7.114532e-15,-2.0676101e-26,-1.2678849e-34,-6.3305344e+18,-2.7625113e-36,-0.00019371489,7.784962e+09,-2.3006628e+30,-7.0658e-18,-1.2537368e+20,-1.5637337e+29,4.381774e-26,-2.1354309e-26,7.6834564e+37,9.545303e+12,2.6922667e-31,1.0968589e-11,1.4238056e+32,6.478207e+24,0.00022710726,5.184652e-37,3.2779668e+06,9.59659e+11,414.1142,5.659327e-36,-1.37759e-34,-7.6373096e-32,-1.6038332e-09,4.1501897e+10,-4.5330802e-21,9.125857e-25,6.691997e+16,-1.3264425e+16,-1.7749392e+22,35.398594,-2.8081838e+26,-3.8716167e+30,-4.8446675e+12,1.2598486e-20,-1.8856188e-16,9.6015775e+27,-7.02611e-16,0.69990265,-1.0601654e-19,-1.4657665e+19,2.7670795e+23,-5.989542e+20,1.0847093e-05,2.3044694e+37,-2.9812768e+29,-3.2893107e-12,-33.196266,-2.3591674e-20,-1.5416242e+26,-6.5232204e-29,2.9791151e+13,-1.02992517e+37,-6.7637524e-16,-6.754042e+36,-6.636439e-19,-2.6693852e-31,-1.5665215e+31,-7.407848e-10,-4.6651164e-20,3.1235548e+07,1.9629187e+15,1.2839874e+32,5.3346138e-08,7.147888e+24,5.847702e+29,-3.1915987e+20,3.2970518e+23,-6.808215e-31,-1.5546383e+32,-2.3927095e-32,4.0065263e-31,-0.29649204,-7.817091e-21,-8.578088e+08,6.550397e+34,-4.3443125e+19,2.7542615e+12,-4.4228447e-30,-1.1971292e+38,-4.6702887e+26,3.4384684e-10,-2.433388e+27,4.877223e-32,1.0352857e-11,-0.00071119546,-127086.66,4.5883062e-05,6.9224357e+21,7.0097214e-25,3.7483234e-14,27905.424,-1.3233843e-28,-1.5671516e-34,-1.5490574e-29,-1.1822398e-26,-1.5754271e-37,3.428062e+21,-8.327511e+27,-3.2472666e-37,1.6437518e-24,12493.995,-4.9383485e+24,7.522739e+07,186640.38,-3.8326196e+21,9.279061e+23,-1.6685216e+07,5.045924e-05,1.2225799e-12,-1.10480045e-07,1.4848774e+17,-2.7090658e-20,1.9479057e+23,-8.864863e-37,1.6154586e+10,2.0292744e-13,5.153773e+33,1.4352609e-22,1.2671873e+27,-2.3806848e+26,-1.3510287e+34,45678.555,730237.3,-6.155532e-20,-6.4340086e-13,-5.4269377e+16,1.408267e-15,5.3908086e+27,1.1247847e+15,3.002213e+09,-1.4919222e+31,2.7663548e-30,3.598979e+23,-1.8102084e+34,9.048947e-30,5203.652,-9.295872e+24,7.9160695e-17,4.24657e+17,7.33754e+15,1.3195474e-30,0.0004468521,-2.1621293e-29,3.025684e-29,6.344137e-27,-3.8254543e+31,-5.7015113e-09,2.8234966e-32,-3.099043e-31,-0.0007217849,21575.338,-1.3881895e-35,9.133829e+14,-1024.9766,-1.8080045e-37,3.092961e+17,-7.799534e-38,-4.669264e+28,6.5846054e-24,-3.4585443e-28,1.3026472e+33,-1.5199914e+18,-8.564141e+22,7.3123614e-13,0.00012696983,1.7701793e+29,-3.321573e-38,5.440605e+15,1.1240135e-05,400467.03,-1.2990331e+06,-8.716011e-34,3.4312407e-08,-4.9634372e-20,9.5578553e+24,5.9711947e-05,-1.6934608e-30,-1.09048834e-23,-3.4048768e-09,-1.38450405e-36,3.6329942e+15,-1.7715137e+28,1.1212521e-31,-1.6485112e-08,-3.96669e-25,-148737.03,-611.03644,-0.049967844,722050.06,1.4525509e-11,-0.004757508,-3.5867076e+29,-2.8334236e-14,0.00014950057,1.8974243e+23,-6.655475e+12,8.0552455e-33,-3.602343e+18,3.921003e+07,7.053465e-16,-0.00068732,2.0174921e+23,-113.68364,-1.799251e+26,2.1572874e+21,-1.6054799e-22,2.8559284e-29,-5.4403903e-24,1.8829607e+11,3.9293067e+15,-5.4045876e-33,1.8607923e+37,9.271587e+27,-1.26815e-14,-1.5798694e+16,-1.0443659e+09,-2.0045465e+33,1.6006747e+34,7.244292e+25,3.5183186e+27,-5.4970525e+20,1.461874e+18,2.038714e-17,9.997089e+13,2.7805977e-34,4.6152108e+07,-259589,-1.561599e+22,-1.7641368e+21,-2.5698854e+24,7.9350184e+27,-5.614582e-20,1.4139597e-19,0.012926011,-806.90045,2.18377e+09,-4.5682364e-06,-1.0019373e-11,-9.4312267e+36,-8.232756e+10,8.0521897e-35,-9.5855705e+27,-4.0678292e+12,2.0280648e-13,-2.960338e-17,1.8505769e-07,-372865.22,3.9683746e-37,-0.00040192503,7.531741e-09,3.4491682e+19,0.04384977,1.1245488e-23,1.3013509e+09,1.054646e-28,-3.4539742e+06,-3.8946708e+31,-2.875654e+10,1.5354944e-29,-2.1998277e+16,-4.988552e-19,-7.8819685e-05,-2.0323907e-36,-3.8438833e+36,-6.1324324e-38,1.3857987e-14,-1.02759116e-19,-3.7773553e+28,1.2115285e+33,-8.7722666e-21,-6.433986e-14,1.958267e+11,3.0698686e-27,-1.6268218e-20,-3.504549e-18,-5.024609e+14,2.9234298e-24,-3.1327398e+28,3991.6948,-2.0440889e-25,-1.8169051e-34,-3.522317e-33,3.185131e-23,7.353024e-17,-2.2593844e+11,-5.9322836e+29,2.3669859e+29,-3.7716948e+07,6.9045724e-12,1.23287894e+36,-1.7350862e+28,-2.4463102e+33,-2.0996942e+33,6.48118e+30,-1.0725687e+33,-2.053883e-39,1.6565287,258361.69,-2.9037657e+27,3.9809908e+17,-6.839905e+08,1.5183219e+36,5.0694654e+32,-3.1612231e-22,2.316999e-35,1622.4696,1.2924275e+19,-13.395993,3.3238469e+28,8.674765e+06,3.5549124e+07,5.0755197e-28,2.7512852e+30,0.0008558714,-3.7883747e+18,-7.6568884e+21,-4.321223e+36,-2.3300378e+13,-2.5671594e-20,1.5445177e+28,-1.2406198e-08,7.261463e+17,-2.329874e+28,1.6979984e-09,1648.9932,-4.3462295e+21,-1.5935611e-19,95.203186,3.029848e-12,-2.3425319e+21,-6490.3726,7.078876e-15,6.621367e-25,1.9671624e-36,8.811079e+29,5.917642e-15,-1.8291024e-19,-1.7264534e-29,2.5669246e-06,1.2450758e+09,-2.8998109e+23,101432.61,-4.2003347e-34,-0.0015121959,-1.3949772e+35,-2.4601503e-23,-1.7320442e+24,-1.10281304e+08,8.4258636e-13,-2.026796e-17,1.1635129e-11,3.5834166e+23,-7.0296214e+22,-6.0035014e+08,-2.9387185e-27,-1.6398929e+09,-1.0181011e-33,-6.311564e+22,7.038771e+20,1.5264363e+30,-1.3027681e+11,1.1590823e-34,-2.6183139e-23,-6.907942e-37,3.2095227e+09,-1.0060934e-37,-8.15462e-25,3.0612058e-29,-7.270971e-05,1.7029327e+09,1.4079879e-28,1.1341949e+36,3.0423907e-37,-7.212576e-25,-5.364587e+12,7.5186526e+20,3.8966607e-07,1.1255939e+14,-7.3194755e-26,-1.8608558e+06,6.845326e+23,-6.3652277e+28,1.2174188e-30,-8.9577663e-35,6.355152e-24,-2.9360056e-24,-3.7761166e-07,-8.925035e-17,6.155995e-36,-3.2674475e+16,6.01666e-21,1.9340822e+25,-1.0541855e-08,1.3527877e+13,9.071381e-29,-2.3711524e+24,-0.0030249585,8.8087186e+29,-7.269473e+14,2.0005875e-09,-9.475562e+19,1.2794023e+31,-2.0927422e+35,1.6837285e-16,-2.5049106e-36,9.042067e-05,1.649178e+24,-9.0291775e-38,-4.1476777e-22,-2.4413687e-06,404014.9,2.6481518e+29,-5.5212383e-24,572.2843,4.6005135e-18,-5.127256e-37,-5.0889496e+12,-7.1179462e+16,2.30567e+34,2.0651227e-23,1.3711946e+29,-7.1656856e+24,3.0988597e+38,-4.0662836e+20,-0.21979055,1.503994e-33,-1.1131797e-30,2.5561101e+11,-3.4350512e-34,-9.123138e+31,-5.000392e-36,14271.518,-1.3223731e-33,-3.8244627e-38,-2.9930296e-17,-0.11207763,-2.0120893e-07,2.3817823e-11,-213.32254,-7.165237e-25,-5.597976e-35,7.43877e+10,6.9071765e-36,-6.393414e-24,-6.2913816e-05,9.216249e-14,-3.6475204e-17,1.3597186e-38,-2.0772864e-23,3.0582033e+22,-3.4489128e-38,2.044839e+34,-173586.2,-1.7325878e+22,-1.1354298e+15,-1.5955651e-19,-5.1043476e-06,3.070723e-36,5.821258e+30,-2.2572621e-07,-3.89677e-05,4.089232e+35,3.6503324e+28,62756.777,-4.598306e+24,0.00020173802,0.00035390237,631804.4,-6.8479664e+28,-5.3282614e-38,-2.5512314,8.317394e+27,2.6341429e-14,-1.8256413e+25,-8.7887223e-16,-0.003676547,-3.5178911e-31,2286.564,-7.7158265e-06,1.4228242e-08,5.3906958e+32,2.8875377e-27,-1.1080453e-23,3.784787e-13,-1.23537596e+33,8.386084e-37,3.6299698e+06,4.3560966e-17,-1.1840245e-35,-54140.23,1.3779113e+07,5.2374735e+13,3.0342788e+13,455.895,1.6457556e+26,-3.0911805e+33,1.0414469e-14,3.5612114e+14,6.532248e+20,-1.1709916e-23,-8.094808e-09,-1.943818e-37,-3.847654e+16,-2.5502352e-23,-1.1559412e+32,-1.427934e-39,-8.315707e-25,656334.44,3.3853377e+34,-5.114119,-2.9730916e-11,-8.506682e+26,-90860.86,-58281.53,6.242738e-33,7.5665884e-13,-2.3129599e+28,22.430443,-6.827717e-33,1.1227465e-33,2.5419736e-11,-1.5369003e-09,-9.226135e-26,9.482052e-10,8.352227e-35,4.9827884e+17,-1.1659414e+25,-6.674202e+12,-0.00028884358,2.1114806e+08,-1.839636e-37,3.2741785e-15,-1.0405002e-19,-5.80038e+24,2.235902e-14,4.8417875e+16,3.779206e-32,-3.936467e+12,1.3787029e+18,-1.7535119e-37,1.6597231e+26,1.46242055e-36,2.6357326e-28,-1.8603381,-2.092685e-22,-6.9043464e+30,3.2264756e+18,1.447243e-18,-1.19534224e-17,0.011875337,-9.269422e-15,-2.977923e+10,2.2271031e+16,1.1870988e-34,-1.0023158e-19,2.419461e-13,8161.9,-9.317509e-25,-1.6346035e+14,8.546791e-33,1.2974216e-21,5.0263483e+16,-1.7699504e+18,-2.0935288e+24,-0.09619193,1.8469096e+06,-1.6789248e+24,-152.83714,-3.843739e+26,-2.1674816e-07,-2.1831715e-11,-6.9348023e-25,-5.0922765e-08,-1.511781e+23,-1.9433844e+24,-1.1762286e-29,-1.4502705e-12,1.8045773e-25,3.1179517e-33,-1.0459524e-31,-7.484386e+33,7.7068844e-11,7.409976e-10,5.046463e-30,8.556721e-14,-4388.7144,-0.8138855,7.910979e-25,8.4195094e+16,-202.3147,-1.21115325e-36,-1.8872508e-10,3.8687866e+24,3.909483e-21,-4.305816e+26,-6.754998e+17,-8.994962e-05,-5.7793534e+21,0.028250126,4.6537604e+19,1.1519273e+26,-10.123301,5.0324147e+25,-5.4325418e-17,-3.3260768e-12,-2.0407294e+10,2.4628652e+12,2.8453346e+21,-7.063029e+06,-3.2588922e-38,1.7463818e+07,-4.2779372e+21,3.2590105e+22,-1.383132e-21,-1.8813328e-26,-2.5532516e+24,2.8405648e+28,2.2541258e-37,5.608257e-19,-8.775791e+09,4.594934e+32,-3.131137e+33,7.7120914e+29,3.299999e+26,1.5985476e+27,7.508715e+15,789.03375,9.406177e-20,7.36966e-16,-9.431699e+36,-0.06561354,2.8131916e+07,-3.3856718e-06,686747.06,-1.0791758e-35,4.4249064e+32,2.2150342e-17,-2.0915023e-19,0.02609501,4.898755e+37,8.178434e+18,-1.56650825e-30,-2.5146896e-09,2.334187e+16,1.4011083e+25,7.3047796e+34,8.093299e-32,-2.503713e+19,1.6829074e-16,5.295867e-37,4.0903262e+14,7.3280665e+24,1.5461425e+11,5.840116e+21,-1.3523108e-36,-0.7218942,-2.701263e-26,3.9198095e+13,3.653263e-09,-1.6278367e+34,-3.462658e-32,-0.04677903,-8.876695e-07,2.8554938e-32,-2.3500232e+06,812.42706,2.4730765e+10,2.5356499e-12,4.0438016e+11,0.3455552,-0.2250181,-2.1117975e+13,-5.1160806e+37,1.2280679e-34,-8.0399994e+21,3.154417e-18,1.0392757e-34,-1.4322928e+20,6.480398e+20,-1.919189e+29,1721.9951,63845.6,1.7009491e+17,-300286.72,2.018023e-09,3.796433e-11,-8.3164514e-10,-5.091692e-30,-1.3286702e-17,-1.5028155e-09,2.1825825e+37,-1.3727429e-34,3.6363902e-28,-12489.18,8.560835e+30,5676.023,-3.9029213e-24,6.707239e-31,-5.105422e+23,8.990932e-33,4.4370435e+14,6.81181e+30,-770006.94,-3.6943378e-19,-75490.43,-2.1685374e-31,1.17972775e-32,-5.9114274e+36,1.2722001e-28,2.7322908e-13,2.0323659e+21,-0.45402563,1.5171535e+37,-0.0001222167,-1.5922759e+32,2.8983976e+14,-2.4112077e-29,5.3024262e-15,3.5643486e+10,-693.5152,-2.311839e+17,-7.718888e-22,-3.7641685e+22,-5.551693e-08,-3.2726888e+17,7.602768e-20,-1.8032012e+25]':::VECTOR,'01010000002AF8A2452849FAC1ACF181407F30F841':::GEOMETRY,(-330470526160741847):::DECIMAL),((-59):::INT8,'[-2.7889768e-37,-4.0510146e+33,-6.675325e+19,1.3383248e+07,-1.679141e-21,2.4317224e+15,-4.3142416e-13,-1.3602022e+22,-2.1235102e-38,-1.4731064e-19,1.267348e+12,1.2558748e+30,2.8090862e+36,1.2517815e+17,5.4647807e-30,-1.3252061e-15,-6.141346e+20,687344.9,0.0008892999,5.8575535e+28,-1.7376811e+24,-0.0008712709,362263.88,-3.816557e+17,1.1805584e+12,-6.590249e-29,-1.681904e+11,6.2595393e-23,6.382495e-32,-0.0003678657,2.2143387e-17,5.5051354e+21,1.654847e+37,-3.793265e-08,7.005663e-06,-9.152121e-09,-2.9553138e-09,1.4515788e+09,54704.406,-4.0735778e-21,2.058142e+07,3.018386e-19,-1.8710067e+34,1.3740502e-05,-5.525283e-19,-1.4862738e+15,1.707863e+11,-1.1078592e+08,-2.0825276e+26,-1.70712e+35,-227.96352,-7.650777e+26,-4.8455947e+33,2.0735004e-35,2.4811123e-18,0.0017315898,-2.6764208e-30,2.646166e+14,1.1204579e+20,2.0520703e+31,-12350.942,-5.575824e-18,-6.0219763e-30,7.5905865e-21,-1.1006156e+33,1.2339718e-05,-3.5560318e+34,-8.061794e-28,1.2915979e-36,4.622961e+13,1.6249733e+26,1.6926399e+06,5.7120657e-24,9.105492e+19,5.7008646e-35,-3.781826e+36,2.1143813e+30,3.17183e-08,2.0013817e-19,0.0031366893,-108614.66,-1.0592394e+25,-2.3754259e+37,8.230975e+08,-1.1971313e-21,-4.2559997e-20,1.0664356e-17,-3.7584142e+11,2.7862328e-38,0.118085876,-6545.682,2.3471331e+13,9.0779796e+17,4.1219457e+25,2.8957492e-17,-1.704465e-34,0.0012293714,1.5717987e+26,3.3818725e+16,-9.3739525e-27,-5.7377914e-36,-2.0708102e+27,-2.7150382e+13,8.092452e+21,-4.8501447e+13,-0.0008846945,0.097763106,6.522698e+12,-4.9726926e+35,-4.763936e+24,9.457563e+20,2.9098242e+28,5.768108e+23,-8.166529e+18,1.4513058e-30,1.3906943e-14,4.031053e-16,-1.145023e-05,5.6459914e+12,-0.45942006,116.55952,-0.32603937,-4.6703365e-17,2.246903e+20,-4.576785e-07,5.7728095e+22,6.6428446e+22,1.845237e-39,6.2866124e+15,1.7273984e-05,6.1452833e-34,5.847374e+08,-1.0662382e+38,-1.0276034e+22,-6.6950415e+30,1.4627001e-35,-5.096816e-18,-1.526359e-23,1.9046215e+28,-1.0046904e-28,1.4859925e+35,-9.978342e+26,7.8868974e-16,-9.656181e-11,2.7227597e-31,8.624298,-2.171024e+31,-1.8043626e-28,-4.8521827e-22,-4.5582904e-18,5.380417e+30,1.8158448e+19,1.1515821e+21,4.5071262e+21,-6.5278708e+07,0.0011656596,-1.0353364e-19,1.9887264e+23,3573.8599,-2.7799575e-13,-1.5034752e-14,-3.157017e-36,-1486.9755,-9.007102e+35,1.8744597e-28,8.645815e-14,-5.7178917e+22,-4.537274e-22,0.9130955,-3.8964186e+08,-2.2966054e+15,-1.4988484e+37,1.9217434e+27,1.2814494e+25,-2.8530686e+26,11422.439,-0.12502642,1.7521917e-33,1.2934098e-11,2.7148641e-34,-3.3690624,1.1652695e-08,-5.985423,5.0042158e+30,-2.0076421e+29,-7.4550136e+15,0.0002386202,-9.834661e+10,0.0020607414,-1.6331447e-05,-1.605845e-19,9.6112595e+35,3.4298072e-08,-2.540525e+20,248273.34,-29.27461,2.3302093e+12,-1.1745297e+31,-9.395553e+08,-9.297072e+31,3.774717e-17,-2.1188668e+22,1.9574715e+09,5.549083e+26,-7.879563e+17,2.0669755e-38,5.981379e+35,1.6048828e+06,-0.028976534,2.4315168e+29,-2.1801264e+26,-2.741585e-33,1.5496696e-10,2.010197e-25,-3.7821424e+36,410.8603,1.3153576e+18,2.2489736e-12,-1.3279278e-25,-2.24803e+19,-3.3847207e+34,-8.0111657e+21,-2.319575e+08,3.2869253e-31,-640781.5,-2.0239064e+24,-1.11724746e-29,7.064724e+37,4.4245047e-23,4.4621266e-29,-15.434979,-1.6781091e+16,5.9726807e-22,-3.7109208e+06,2.5583328e-17,-1.0459153e+19,6.389608e+18,7759.069,-4.2346934e-38,-2.3620243e+34,-1.2254715e-33,-1.4726119e+22,-1.1425589e-14,2.8322704e-37,-4.3584424e+18,-2.6021247e-14,-2.5344138,1.9514247e-35,9.4891654e+24,-2.4010294e+20,4.2482562e+33,3.326252e-37,1.5777118e-21,-4.892029e-30,1.5742434e+32,-2.760206e+35,0.17936757,1.5350881e+26,-0.5220448,-5.276246e-08,1.0130397e+14,-9.0167276e-08,2.7987168e-09,3.9240294e-07,4.962633e-22,4.1862036e-17,1.2953524e+26,-2.0062046,-7.321878e-06,4.8833043e-35,-1.04906535e-22,2.7371341e-27,-1.1703896e+31,-3.1026494e-29,5.3487586e-17,9.514387e+27,1.4587271e-19,1.8000644e-11,-4.5092052e-35,4.4684637e+17,-7.236019e-23,2.7500116e-29,0.01276934,-5.713197e+29,1.6537176e+18,3.958609e-23,1.3930996e-24,2.2687583e-07,2.2652142e-07,-3.760972e+10,1.13260464e-10,-4.625686e+31,3.4328106e-11,192.11536,-2.115342e+06,-1.6486153e-38,1.8851415e-28,2041.0328,-2.1317241e+18,-7.1260485e-18,3.338864,2.7228449e+32,6.3512584e+24,1.5390744e-25,4.034411e+26,1.0756904e+20,5.4273115e+13,-6.494458e-09,-4.6457603e+24,-2.3436684e-12,2.2623414e-20,-612.2786,-8.2997234e+24,-4.7159177e+17,2.2838589e-29,13.965542,-1.04644004e-29,8.887971e+13,0.8994399,-6.7184256e-20,-9.718009e-09,0.12644044,3.887495e+16,0.44541,-1.6087393e+28,7.80313e+16,9.919982e+31,-2.6760592e+14,26359.951,-9.749914e+20,7.67329e-08,1.4154529e-07,-2.450707e+07,1.8651867e+34,-1125.5985,-5.8791003e-09,-717.40125,0.11429084,1.0562646e+06,3.1023226e-06,-3.3136024e+07,7.8154934e+37,-3.0427577e+21,356.07883,1.6007798e+14,8.0040116e-33,18898.443,-1.2794438e-25,-6.296312e-05,-1.8111856e+24,-6.0117094e-35,7.764972e-21,0.45500392,15.182077,-0.00019490761,-5.1828123e-27,-67888.29,5.988478e+35,3.514073e-07,-5.1740077e-23,-1.5654224e+29,213854.17,1.3058347e-09,-3.898589e+33,6.872821e+18,-3.2329107e-17,2.4060141e-20,-1.1365506e-36,-7.77175e+21,2.4475994e-22,47.107056,8.119536e+29,-2.5253627e-07,-2.2394394e-21,8.549638e+27,4.901362e-38,-2.813182e-14,3.3451669e+38,-7.336963e+18,7.349468e-37,1.7754901e-25,2.0233737e-27,3.7726556e-36,-7.9683737e-25,-1.13754135e+23,1.2135686e+26,2.3482223e-28,-4.429529e+35,5.3365174e+29,-1.6106261e-18,14.351335,6.4389947e+25,5.6425903e+18,-3.7118992e-31,-6.7542326e-33,3.1895446e-29,3.3626906e+29,0.00037803818,1.5596832e+18,-1.8166614e+17,-4.1794305e-18,1.5178641e-31,2.572048e+19,-0.27842376,-6.3544404e+22,7.623904e+21,-1164.9175,0.029311782,-3.281501e-13,0.00015802607,6.384057e+06,30283.838,1.4678887e-11,1.6190584e+09,1.1022137e-05,-1.0310357e-34,-2.9503777e-32,4.7109658e+24,-6.805602e-16,-0.92278147,-3.908936e+08,0.04699782,3.5286312e+06,6.279322e-19,1.7647976e-25,2.745043e-18,2.6065073e+21,1.0706548e+11,-1.2987091e+35,9.6990784e+14,1.7347089,2.6821646e-05,-3.3026874e-13,-2.3172275e+21,-3.730054e-19,-1.9625442e+13,-1.6732235e+36,-2.4051135e-16,-1.5828572e+07,-3876.8015,2.588316e-34,1.456342e-11,1.5308187e+36,-1.4785061e-27,-1.7836839e-09,1.2303223e-35,-1.6826651e+13,2.466358e+22,-103.29153,-2.5845093e-09,-7.864183e-06,1.8255603e-14,7.3782124e+09,-3.4335974e-12,4.9204024e+09,-2.4737331e-14,-0.058724105,1.1441128e+33,6.305187e-09,-3.428215e-16,2.7741617e+34,1.0982296e-32,7.6079614e-20,-1.8256581e+19,-7.1130235e-09,-8.590241e-22,1.1487581e-16,2.3630374e-20,6.264649e+31,5.8605094e+08,2.3737545,2.2567317e-34,968372.8,-4.505952e+26,1.4677413e-38,-2914.0715,9.448015e+28,2.1251215e+37,1.4110284e+26,-3.7000234e-31,-1.4737005e+23,-9.532945e+20,-3.3166632e+35,-7.398003e-23,-4.6472084e-37,3.834933e+13,-4.414249e-34,1.9949313e+37,2.6056863e+11,-1.7096118e-27,-2.1488821e-21,1337.8322,-3.8950215e-23,2.1467077e-25,-5.895914e-28,0.39643085,6.26275e-32,3.9567696e-10,-0.0025995886,-3.0162684e-24,6.104711e+28,4.1127695e+09,-5.9812373e-32,2.2145062e+33,7.261139e-08,-2.7139525e-21,-1.3737264e+26,0.21784888,2.1820738e+26,-3.63857e-36,1.3661854e+10,-1.4022266e+30,3.2122714e-06,1.0706993e-14,2.6333215e+23,9.962437e+20,3.0824486e+24,1.15423e+13,1.25038455e+17,-5.0918434e-23,-5.379777e+15,1.9578841e-37,-2.530936,-2.3697379e+11,-2.2614085e-23,0.5304916,0.31200254,-5.914755e-30,-5.6994535e-25,3.0599546e-28,-2.9222774e+30,3.2047068e-26,-2.6395621e-33,1.9852706e-10,0.007504515,21735.611,2.2227611e+30,-4.2158573e-19,6.731321e-24,-1.5906618e+33,-3.0681813e+19,2.9590761e+16,2.8655245e+09,1.5476496e+15,1.7777105e+13,-5.191718e-22,-1.9437005e+10,-0.014457941,1.0162842e-13,9.181913e-31,2.3046753e-24,-6.265602e-25,7.7047885e+08,-1.0126689e-05,-1.9287812e+10,3.4703272e+19,-8.678714e-10,25.658554,1.6828871e-22,-5.59689e+16,-7.3289077e-35,4.484245e+23,-4.2789107e+33,-1.1377256e+22,-2.3437448e-28,5.924606e-31,3.7717772e-22,17.680195,1.1313357e-28,8.093915e-34,2.1046233e-35,-2.5275063e+38,-5.602037e+30,1.1402414e-18,3.5773194e-32,-170.66321,-4.3359605e+29,4.402552e+37,3.3491336e-37,-4.6419275e-12,-7.290053e-34,-0.0029138366,-3.3781788e-14,-6.688095e-22,-0.00021626035,2.8217273e-10,3.661078e-13,-8.551032e+22,4.0885877e-35,-5.731857e-31,5.710454e+35,-4.7003166e+14,1.2521822e-28,8.73475e-24,9.703146e+34,-4.834238e+22,-1.2802739e+36,-1.2572021e-15,3.586497e-10,2.1373673e+36,1.1457073e+08,3.2905453e-24,-1.140267e+38,1.0434675e+09,1.0747329e+29,-1.059468e-16,-1.24795e+37,2.0417655e+20,-3.9467998e-25,-7.726608e+31,5.6469604e-23,1.1955738e+08,1.6210376e-20,5.4981397e-06,-6.543894e-07,-6.564542e+11,1.5654395e+23,1.6583517e+13,0.027022682,-5.6749745e-23,1.6849339e-09,-8.558956,-6.407768e-31,1.1167883e-15,-2.2359243e+36,9.303381e-38,3.055255e+29,1.1748397e-22,5.460029e-32,8.1739125e-35,-5.422013e-07,-1.6886044e+36,1.0705046e-37,4.3930516e+12,3.0037921e+16,-1.5081888e+17,2.0232245e+14,2.8355395e+13,1.6585464e-06,-3.185561e-26,5.832764e+25,-3.5176466e-33,4.6806112e-30,3.36147e-39,-0.10711979,2.309501e+23,0.000112512185,2.629083e+06,7.2405995e+14,1.7076585e-09,-1.3783946e+10,4.99797e-38,5.048091e-31,-135612.98,-8.747065e-37,-7.908919e+17,4.091651e+29,9.495302e-08,1.0548696e-06,-2.3484438e+38,-7.2157955e+34,-2.2766866e-18,-1.9325155e+18,9.610231e-17,3.4674966e-10,6.646413e-24,4.027044e-11,5.3097382e+23,7.389363e+29,7.494107e+13,2.4888848e+28,-8.533028e-16,1.09394905e+21,-6.058355e+06,6.655714e-34,-2.9627436e-24,7848.062,6.9574077e+25,-6.240719,-8.986721e-11,-1.7380397e+18,5.1363487e+15,-2.7157125e-35,4.7511733e-30,2.2829218e-25,5.4518756e+07,-1.1903644e-35,2.0594804e-17,2.008849e-13,-7.323122e-32,3.9457063e+34,-4.1227813,6.6341533e-32,4.1452525e-37,-1.1023877e-19,8.700098e+37,-9.078407e-39,13317.843,-7.2493634e+25,-1.7915861e-31,2.982676e+22,-4.7690236e-08,-2.6064097e+12,-0.026007682,-9.308467e-25,-9.1924354e-05,4.3147663e-36,-1.3588499e-12,-2.042099e-06,-3.889758e+21,1.4853598e-19,-2.7787695,-2.0825096e+29,-2.023225e-29,-6.269957e+25,-1.4445459e+35,-2.1280939e-29,3.0079977e-27,-3.9774693e+37,-1.3818675e-32,1.8521708e+35,2.0117514e-35,6.0634814e+32,3.422491e+26,3.224963e-30,2.6977655e+10,-1.5536149e-09,1.9142853e+18,4.8822797e-18,-1.0960905e+17,6.8899664e-09,-3.3004652e+38,-6.834875e-10,1.1126437e-27,7.3908955e+27,-2.8425036e+18,-1.0326343e-25,-8.4220104e-30,2.8418823e-13,-19.402737,1.3392015e-30,-0.015855957,-4.0437006e+10,-4.609021e-38,1.3915184e-06,4.975698e-28,-2.2206637e-10,-1.23934876e+11,-1.1832867e-09,6.774656e+29,-4.281884e+36,2.9041985e-14,-9.739537e-34,7.3713194e+12,8.8490645e+15,-1.4337577e+37,-3.4583804e+23,3.5923027e-18,-85.73696,3.0254034e-21,-7.396942e-05,-1.015138e+15,97721.68,-3.3782228e+27,2.0607925e-20,-5.369043e+16,-7.1588955e-31,1.2387531e+10,8.541224e-07,3.1353585e-26,1.973594e+14,-4.6426862e+14,0.031133981,1.3749395e-14,-1.25004365e-33,-5.942463e+21,-1.3071452e-21,-1.113794e-33,2.1789979e-09,-4.6336955e+36,6.2768773e+31,-1.8694595e-36,2.9096227e+37,-6.296386e+34,-1.7766567e+33,1.9291142e+16,2.2008027e-15,-7.307431e+12,8.679964e+14,1.14526666e-10,-4.3686353e-26,8.385655e+26,1.6151351e-26,-0.0005426209,6.443812e+19,3.141642e+08,-1.2479057e+08,1.9326013e-19,-1.0998847e-34,2.9276625e+10,-2.2247327e+26,5.0356807e-36,-3.8053868,4.2700596e-33,-1.5616518e-11,5.9035273e-27,-2.1498842e-32,5.160075e+26,2.350235e-17,7.665928e+30,3.6934382e-22,1.8272535e+21,-6.436703e-08,1.0288743e+10,8613.814,-1.253976e+15,-3.2851456e-34,6.0493633e+26,-2.7272866e-24,1.1220613e-26,2.6357381e-23,3.6037682e+13,-2.0098351e-17,-8.3686864e+33,-2.8894107e-29,8.430772e-09,-3.1764108e-36,-1.0315002e+18,3.1362276e-14,-2.544202e-24,-4.7210586e+10,-1.6548252e-23,-3.9752866e+32,-5.334426e-25,1.0040571e-14,-1.0385244e+11,7.2278e-39,1.3907629e+17,2.5062619e-09,-6.3641947e+35,-1.6160442e-19,1.4280978e+14,-5.7065758e-33,-8.020991e+37,-4.145709e-34,-1.1709328e+14,-2.0160106e+06,1.5438216e-30,-1.0611992e-18,-1.5872213e-23,0.16278026,1.079973e-33,6.401741e-26,2.939347e+25,1.6812281e-08,-1.7799993e-12,-0.00017600282,173270.73,-6.396702e-17,1.4115351e-16,2.174178e-17,1.06634005e+30,5.024065e+07,4.3331e-11,6.458163e-11,-1794.9257,3.8220758e+06,-5.278943e-11,3.4121797e-38,7.776878e-20,2.2006066e-28,-1.2228796e+29,2.275948e-08,9.463824e+11,-5.8729682e-08,4.178641e-28,17.56325,-962415.2,1.6944024e+26,-4.0648415e-38,-1173.0435,5.907776e+35,4.1979475e+33,-1.3883172e+24,7.4826326e-11,1.0909703e+31,-5.218587e-16,1.5166166e-09,-6.247063e-39]':::VECTOR,'{\"9RfZ\": [{\"baz\": null, \"zIn\": false}, null, false, true], \"Tt2RCnh\": 1.0127436346937484, \"foo\": false}':::JSONB,'17:43:12.854452-01:34':::TIMETZ,true,'''CcwLRmjHe'' ''PybimW'' ''QxUuccntAE'' ''bcrUMWIr'':273C,277A,371B ''dGAuzeKb'':578C ''dYmcu'' ''ewyINoO'':385C,821B,991B ''uEzc'':751B,945B':::TSVECTOR,'[-2.0335705e-13,-1.23869e+12,5.0158158e-17,3.015393e-38,-1.1430671e+27,7.1629495e-32,-7.096858e-21,-0.000875772,2.156195e-08,1.1826801e+26,3.896476e+27,-3.3449521e-12,-2.3253266e+11,1.6047474e-17,5.6428367e+32,7.280408e-33,5.0583235e+34,3.146448e-30,1.6489811e+06,-5.936247e-07,9.650402e+17,1.5578771e+33,1.185901e+27,-1.0797248e+25,-1.00606764e+21,-1.1531769e-23,-3.681473e+08,3.9120686,2.190006e-06,1.1840731e-14,-6.101732e+11,3.8274536e-18]':::VECTOR,'010400008000000000':::GEOMETRY,(-38572389.25432331467):::DECIMAL)",
   "potentialExecErr": "23502,23505,23514"
  },
  "COMMIT"
 ],
 "expectedExecErrors": "",
 "expectedCommitErrors": "",
 "message": ""
}
    schema_change_external_test.go:134: 
        	Error Trace:	pkg/ccl/testccl/workload/schemachange/schemachange_test_test/pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:134
        	Error:      	Received unexpected error:
        	            	***UNEXPECTED ERROR; Failed to generate a random operation: ERROR: column "crdb_internal_column_3_name_placeholder" does not exist, referenced in "crdb_internal_column_3_name_placeholder IS NOT NULL" (SQLSTATE 42703)
        	            	(1) forced error mark
        	            	  | "fatal error when running txn"
        	            	  | github.com/cockroachdb/errors/withstack/*withstack.withStack::
        	            	Wraps: (2)
        	            	Wraps: (3) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).runInTxn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/schemachange.go:490
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).run
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/schemachange.go:587
        	            	  | pkg/ccl/testccl/workload/schemachange/schemachange_test_test.TestWorkload.TestWorkload.func4.func6
        	            	  | 	pkg/ccl/testccl/workload/schemachange/schemachange_test_test/pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:123
        	            	  | golang.org/x/sync/errgroup.(*Group).Go.func1
        	            	  | 	golang.org/x/sync/errgroup/external/org_golang_x_sync/errgroup/errgroup.go:78
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1695
        	            	Wraps: (4) ***UNEXPECTED ERROR; Failed to generate a random operation
        	            	Wraps: (5) ERROR: column "crdb_internal_column_3_name_placeholder" does not exist, referenced in "crdb_internal_column_3_name_placeholder IS NOT NULL" (SQLSTATE 42703)
        	            	Error types: (1) *markers.withMark (2) *schemachange.ErrorState (3) *withstack.withStack (4) *errutil.withPrefix (5) *pgconn.PgError
        	Test:       	TestWorkload
Schema Workload Stats
Total Schema Statements Executed = 24
Total Schema Statements Succeeded = 24
Total Schema Statement Expected Failures = 0
Total Transactions Committed = 14
Total Transactions Rolled Back = 11
Total Transactions Executed = 25
    schema_change_external_test.go:99: backup, tracing data, and system table dumps in /var/lib/engflow/worker/work/1/exec/bazel-out/k8-fastbuild/testlogs/pkg/ccl/testccl/workload/schemachange/schemachange_test/run_8_of_25/test.outputs/logTestWorkload1025672606
    panic.go:626: -- test log scope end --
test logs left over in: outputs.zip/logTestWorkload1025672606
--- FAIL: TestWorkload (8.39s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=8</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #131002 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot T-sql-foundations branch-release-24.1.5-rc release-blocker]
- #130283 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot P-2 T-sql-foundations branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestWorkload.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

pkg/ccl/testccl/workload/schemachange/schemachange_test.TestWorkload [failed](https://tanzanite.cluster.engflow.com/invocations/default/94b55381-fdb2-4698-8201-4ed37a9f0c8c?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL3Rlc3RjY2wvd29ya2xvYWQvc2NoZW1hY2hhbmdlOnNjaGVtYWNoYW5nZV90ZXN0) on master @ [a055dec86299452f1951416c16f5938e523f8559](https://github.com/cockroachdb/cockroach/commits/a055dec86299452f1951416c16f5938e523f8559):


```
     [
      "public",
      "table_w1_152",
      "col\r15 2_w1_167",
      "col152_w1_154"
     ]
    ]
   },
   {
    "query": "SELECT (false = ANY (ARRAY[true = true, false = true])) AND (count(*) = 0) FROM public.table_w1_152 WHERE \"col\r15 2_w1_167\" = (true)",
    "result": true
   }
  ],
  "previousStatements": [
   "INSERT INTO public.table_w1_152 (col152_w1_153, col152_w1_154, c😠ol152_w1_155, \"col15%p2_w1_156\", col152_w1_157, col152_w1_158, \"col152\t_w1_159\", col152_w1_160, \"col1%v52_w1_161\", \"col1\"\"52_w1_162\", col152_w1_163, \"col\u000b152_w1_164\", \"col%q152_w1_165\", col152_w1_166, \"col\r15 2_w1_167\", \"c-ol152_w1_168\") VALUES ('ad2eaaa5-7b68-4062-9152-94ac98e843dd':::UUID,true,'''MYl'':385B,850B ''NoXB'':693C ''WKIN'' ''YBqZfBh'' ''qMpwbKuSb'':81B,117C,487B,694A':::TSVECTOR,'01020000C007000000002E17B3543E7AC16B667040E9FAF0C1FB68BF43AC1EF6C1D271B3CD217B004240C7B7C018ABAC414C28AD80B3A0FBC1FE46AB2B3793014230A70EE93D34DB4148FEDC57CC8DF541C02FD5582A2AFCC1F25B68298DBEF0C100764F825E6DA241449650A79708F8411018B4A04D6BEDC160AAE2CF7F40FF4100856D7EFB73C0413AF131576D76F241C02694DC858CC8C11021D38B9A38E7C1C2B016F8529C02C218A1DBC2177FFA4160A69E815855D84118EEE1958E50D441AAD1ECBC811CF141DC3EC46DF2D6E541F062FEDBDE66FB415ABD7942262BE3C1FEE6E915A44F0042':::GEOMETRY,'kN':::REFCURSOR,(-17):::INT8,'07:56:32.385899+10:12':::TIMETZ,99:::INT8,e'\\x186\\x0fx\\x12={Q\"'::VARCHAR,'08:29:33.79432':::TIME,'2017-07-27 19:25:37.0007':::TIMESTAMP,'{\"9ngB42bAs\": [true, {\"foo\": false}, null], \"Jqf\": {\"c\": null, \"hghyeZH4\": true}, \"a\": {\"baz\": 1.639019729535775}, \"baz\": null, \"nVLi\": {\"baz\": null}}':::JSONB,'1993-04-19':::DATE,'{\"ZsujQ\": null, \"b\": null, \"bar\": \"cNv\", \"ho2UaaD6\": [[0.45648654080313844], [\"rjkHZzFc\"], [{\"bar\": true}, false, true], \"bar\", true]}':::JSONB,true,(-89):::INT8),('049eb20f-14e1-4be5-89db-efc5a3ae0ea2':::UUID,false,'''DQl'':427C,855C,943A ''NrClFbJpr'' ''T'':682B ''TfvJQoPEl'':976C ''Ug'':17A,106C,241C ''bDXoomyhxm'' ''euLn'' ''iWI'' ''kYJi'':151C,610B ''yWXlvqfdg'':173C,581C,984B':::TSVECTOR,'0102000080030000009A08A259A36BFEC154AFC31EF951DFC1ECE42299077FFE413AA504515B97EAC100D49B8DD09DECC1AC5D182BD77602C2F892116E3ED0D5C1CEB41E3C4F2A00C2D8D5B126B857D241':::GEOMETRY,e'\\\\p#\\x17\u0026=':::REFCURSOR,122:::INT8,'00:00:00+15:59':::TIMETZ,(-112):::INT8,'j'::VARCHAR,'02:05:44.922628':::TIME,'1970-01-11 14:50:00.000211':::TIMESTAMP,'{\"Cs\": \"a\", \"Qsb\": false, \"l4F\": false}':::JSONB,'1985-06-23':::DATE,'[[[\"d7hmNL\", false, true], false], 0.18565591177733592, false, false, false]':::JSONB,false,(-19):::INT8)"
  ]
 }
}
    schema_change_external_test.go:134: 
        	Error Trace:	pkg/ccl/testccl/workload/schemachange/schemachange_test_test/pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:134
        	Error:      	Received unexpected error:
        	            	***FAIL; Failed to receive a commit error when at least one commit error was expected
        	            	(1) forced error mark
        	            	  | "fatal error when running txn"
        	            	  | github.com/cockroachdb/errors/withstack/*withstack.withStack::
        	            	Wraps: (2)
        	            	Wraps: (3) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).run
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/schemachange.go:672
        	            	  | pkg/ccl/testccl/workload/schemachange/schemachange_test_test.TestWorkload.TestWorkload.func4.func6
        	            	  | 	pkg/ccl/testccl/workload/schemachange/schemachange_test_test/pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:123
        	            	  | golang.org/x/sync/errgroup.(*Group).Go.func1
        	            	  | 	golang.org/x/sync/errgroup/external/org_golang_x_sync/errgroup/errgroup.go:78
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1695
        	            	Wraps: (4) ***FAIL; Failed to receive a commit error when at least one commit error was expected
        	            	Error types: (1) *markers.withMark (2) *schemachange.ErrorState (3) *withstack.withStack (4) *errutil.leafError
        	Test:       	TestWorkload
Schema Workload Stats
Total Schema Statements Executed = 100
Total Schema Statements Succeeded = 100
Total Schema Statement Expected Failures = 0
Total Transactions Committed = 66
Total Transactions Rolled Back = 43
Total Transactions Executed = 109
    schema_change_external_test.go:99: backup, tracing data, and system table dumps in /var/lib/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/testlogs/pkg/ccl/testccl/workload/schemachange/schemachange_test/run_1_of_25/test.outputs/logTestWorkload3742668641
    panic.go:626: -- test log scope end --
test logs left over in: outputs.zip/logTestWorkload3742668641
--- FAIL: TestWorkload (31.82s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #131002 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot T-sql-foundations branch-release-24.1.5-rc release-blocker]
- #130283 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot P-2 T-sql-foundations branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestWorkload.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

pkg/ccl/testccl/workload/schemachange/schemachange_test.TestWorkload [failed](https://tanzanite.cluster.engflow.com/invocations/default/265291b2-cc9d-41f0-9be3-ebd349e0b7b5?testReportRun=24&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL3Rlc3RjY2wvd29ya2xvYWQvc2NoZW1hY2hhbmdlOnNjaGVtYWNoYW5nZV90ZXN0) on master @ [b6c13686495bbe9ad476b28033461ef7628e18a8](https://github.com/cockroachdb/cockroach/commits/b6c13686495bbe9ad476b28033461ef7628e18a8):


```
    "result": false
   },
   {
    "query": "SELECT count(*) FROM public.table_w1_30",
    "result": 18
   }
  ],
  "previousStatements": [
   "ALTER TABLE public.table_w1_30 ADD CONSTRAINT table_w1_219_col219_w1_227_table_w1_30_col30_w1_31_fk FOREIGN KEY (col30_w1_31) REFERENCES schema_w1_161.table_w1_219 (col219_w1_227) ON DELETE CASCADE ON UPDATE CASCADE"
  ],
  "usesDeclarativeSchemaChanger": true
 }
}
    schema_change_external_test.go:134: 
        	Error Trace:	pkg/ccl/testccl/workload/schemachange/schemachange_test_test/pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:134
        	Error:      	Received unexpected error:
        	            	***UNEXPECTED ERROR; Received an unexpected execution error.: ERROR: current transaction is aborted, commands ignored until end of transaction block (SQLSTATE 25P02)
        	            	(1) forced error mark
        	            	  | "fatal error when running txn"
        	            	  | github.com/cockroachdb/errors/withstack/*withstack.withStack::
        	            	Wraps: (2)
        	            	Wraps: (3) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*opStmt).executeStmt
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/operation_generator.go:3178
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).runInTxn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/schemachange.go:499
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).run
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/schemachange.go:587
        	            	  | pkg/ccl/testccl/workload/schemachange/schemachange_test_test.TestWorkload.TestWorkload.func4.func6
        	            	  | 	pkg/ccl/testccl/workload/schemachange/schemachange_test_test/pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:123
        	            	  | golang.org/x/sync/errgroup.(*Group).Go.func1
        	            	  | 	golang.org/x/sync/errgroup/external/org_golang_x_sync/errgroup/errgroup.go:78
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1695
        	            	Wraps: (4) ***UNEXPECTED ERROR; Received an unexpected execution error.
        	            	Wraps: (5) ERROR: current transaction is aborted, commands ignored until end of transaction block (SQLSTATE 25P02)
        	            	Error types: (1) *markers.withMark (2) *schemachange.ErrorState (3) *withstack.withStack (4) *errutil.withPrefix (5) *pgconn.PgError
        	Test:       	TestWorkload
Schema Workload Stats
Total Schema Statements Executed = 176
Total Schema Statements Succeeded = 176
Total Schema Statement Expected Failures = 0
Total Transactions Committed = 120
Total Transactions Rolled Back = 76
Total Transactions Executed = 196
    schema_change_external_test.go:99: backup, tracing data, and system table dumps in /var/lib/engflow/worker/work/1/exec/bazel-out/k8-fastbuild/testlogs/pkg/ccl/testccl/workload/schemachange/schemachange_test/run_24_of_25/test.outputs/logTestWorkload3537561282
    panic.go:626: -- test log scope end --
test logs left over in: outputs.zip/logTestWorkload3537561282
--- FAIL: TestWorkload (31.72s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=24</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #131002 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot T-sql-foundations branch-release-24.1.5-rc]
- #130283 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot P-2 T-sql-foundations branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestWorkload.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

pkg/ccl/testccl/workload/schemachange/schemachange_test.TestWorkload [failed](https://tanzanite.cluster.engflow.com/invocations/default/4f634b6f-92bd-438a-9d06-0436686c3506?testReportRun=13&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL3Rlc3RjY2wvd29ya2xvYWQvc2NoZW1hY2hhbmdlOnNjaGVtYWNoYW5nZV90ZXN0) on master @ [b6c13686495bbe9ad476b28033461ef7628e18a8](https://github.com/cockroachdb/cockroach/commits/b6c13686495bbe9ad476b28033461ef7628e18a8):


```
     true,
     true,
     true,
     true,
     true,
     true,
     true,
     true,
     true
    ]
   }
  ],
  "previousStatements": [
   "ALTER TABLE public.table_w1_6 ALTER PRIMARY KEY USING COLUMNS (col6_w1_9)"
  ]
 }
}
    schema_change_external_test.go:134: 
        	Error Trace:	pkg/ccl/testccl/workload/schemachange/schemachange_test_test/pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:134
        	Error:      	Received unexpected error:
        	            	***UNEXPECTED COMMIT ERROR; Received an unexpected commit error: ERROR: transaction committed but schema change aborted with error: (23505): failed to ingest index entries during backfill: duplicate key value violates unique constraint "table_w1_6_pkey" (SQLSTATE 23505)
        	            	(1) forced error mark
        	            	  | "fatal error when running txn"
        	            	  | github.com/cockroachdb/errors/withstack/*withstack.withStack::
        	            	Wraps: (2)
        	            	Wraps: (3) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).run
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/schemachange.go:658
        	            	  | pkg/ccl/testccl/workload/schemachange/schemachange_test_test.TestWorkload.TestWorkload.func4.func6
        	            	  | 	pkg/ccl/testccl/workload/schemachange/schemachange_test_test/pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:123
        	            	  | golang.org/x/sync/errgroup.(*Group).Go.func1
        	            	  | 	golang.org/x/sync/errgroup/external/org_golang_x_sync/errgroup/errgroup.go:78
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1695
        	            	Wraps: (4) ***UNEXPECTED COMMIT ERROR; Received an unexpected commit error
        	            	Wraps: (5) ERROR: transaction committed but schema change aborted with error: (23505): failed to ingest index entries during backfill: duplicate key value violates unique constraint "table_w1_6_pkey" (SQLSTATE 23505)
        	            	Error types: (1) *markers.withMark (2) *schemachange.ErrorState (3) *withstack.withStack (4) *errutil.withPrefix (5) *pgconn.PgError
        	Test:       	TestWorkload
Schema Workload Stats
Total Schema Statements Executed = 20
Total Schema Statements Succeeded = 20
Total Schema Statement Expected Failures = 0
Total Transactions Committed = 15
Total Transactions Rolled Back = 10
Total Transactions Executed = 25
    schema_change_external_test.go:99: backup, tracing data, and system table dumps in /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/testlogs/pkg/ccl/testccl/workload/schemachange/schemachange_test/run_13_of_25/test.outputs/logTestWorkload3987417450
    panic.go:626: -- test log scope end --
test logs left over in: outputs.zip/logTestWorkload3987417450
--- FAIL: TestWorkload (10.64s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=13</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #131002 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot T-sql-foundations branch-release-24.1.5-rc]
- #130283 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot P-2 T-sql-foundations branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestWorkload.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

pkg/ccl/testccl/workload/schemachange/schemachange_test.TestWorkload [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17110667?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17110667?buildTab=artifacts#/) on master @ [74333311616b937fea6a995462215a1cb5962686](https://github.com/cockroachdb/cockroach/commits/74333311616b937fea6a995462215a1cb5962686):


```
    "result": false
   },
   {
    "query": "SELECT ARRAY[parent.table_schema, parent.table_name, parent.column_name, child.column_name] FROM (SELECT conname, conkey, confkey, conrelid, confrelid FROM pg_constraint WHERE (contype = 'f') AND (conrelid = 'schema_w0_41.table_w1_161'::REGCLASS::INT8)) AS con JOIN (SELECT column_name, ordinal_position, column_default FROM information_schema.columns WHERE (table_schema = 'schema_w0_41') AND (table_name = 'table_w1_161')) AS child ON conkey[1] = child.ordinal_position JOIN (SELECT pc.oid, cols.table_schema, cols.table_name, cols.column_name, cols.ordinal_position FROM pg_class AS pc JOIN pg_namespace AS pn ON pc.relnamespace = pn.oid JOIN information_schema.columns AS cols ON ((pc.relname = cols.table_name) AND (pn.nspname = cols.table_schema))) AS parent ON ((con.confkey[1] = parent.ordinal_position) AND (con.confrelid = parent.oid)) WHERE child.column_name != 'rowid'",
    "result": null
   }
  ],
  "previousStatements": [
   "INSERT INTO schema_w0_41.table_w1_161 (col161_w1_162, col161_w1_163, \"col16%p1_w1_164\", col161_w1_165) VALUES ('[0.010906358,-5.9243373e+10,7.084823e+19,-1.03249656e+08,-2.8124546e+12,-4.4765278e-17,-1.73296e-27,-0.0035417099,2.0375604e+36,1.2787155e-12,3.4163988e+25,1.5803806e+17,-1.1672625e-07,2.726513e+26,4.5928342e-08,1.2996659e+09,251466.52,-5.9620776e+07,5.1807853e+08,-4.666469e-14,-1.5303094e-27,-2.3815865e-26,54.3509,9.31719e-32,-2.5990997e-24,-7.3427955e+21,1.2296804e-24,-1.4226286e+38,-8.988851e+12,-3.1255075e+18,-1.7843191e+23,-4.8409193e+13,-1.257796e-29,-7.512183e+35,2.3604284e-36,-7.860379e-28,238799.88,3.0759867e+25,-7.2774818e-31,1.7941514e+08,8.810898e-14,8.284129e-16,1.6949618e-32,2.6256486e-19,1.2411612e+17,-1.2439723e+14,4.0698406e+16,6.4441664e-32,-2.9634659e-33,6.7140816e-28,6.986698e-15,3.5743068e+25,4.358001e+31,-2.8506646e+13,3.296366e+15,2.6548405e-05,0.115274884,1.2661924e+17,-0.0012369849,8.757248e+27,-1.3135674e-26,-5.0816863e+31,-2.4833853e-26,4.3742642e-05,4.7322063e+26,-2.5249098e+09,-1.7430312e-11,-1.9943039e+20,-1.7345762e+20,-1.8266286e+19,-4.5880669e+18,1.0797161e+12,1.8449008e+18,6.0732526e-07,-3.918653e+26,-1.092111e+27,-1.4526039e-16,2.156974e-12,1.7351955e+36,1.9645476e-36,-0.028368315,-3.5197424e+07,1.1760813e+30,-3.9314907e+10,3.528082e-22,15193.487,-1.928594e+15,1.6139962e+32,-357.40976,-4.6959725e+23,-8.825821e+09,-1.520672e-10,-3.56665e-33,-3.1778111e-28,3.7190456e-30,1.02302946e-25,3.249462e+37,-6.122844e+16,4.4449653e-06,-2.3988785e-33,-1.1052476e+24,-2.3602047e-26,-6.003299e+14,-1.9265362e-21,-1.3426337e-08,-1.5053589e-24,-5.8145796e-30,-3.184449e-05,-1.04854804e-19,-8.776499e+18,1.1987002e+11,-8.905472e+07,2.520991e-15,2.5171069e-27,-9.325332e-23,-1.4372789e-18,1.231311e-11,1.2540967e-05,6.7882e+25,-0.3328182,-7.226682e-28,8.096202e+24,6.3879444e-12,-9.240584e+26,-1.5644272e+15,-1.1656291e-32,0.00082824664,-1.0573175e-15,2.777379e+22,0.00012046653,-1.8154507e+30,-8.2180386e-18,-9.037412e+14,2.2888668e+11,-3.3711924e-05,-1.0343739e+36,-1.1958092e-14,3.5938274e-06,6.7159655e+24,2.0594965e+16,-1.2838493e+20,-2.7823505e-24,1.4375987e+09,2.9843281e+37,-1.2570222e+22,-4.883505e-16,7.591967e+28,1.678458e-09,-2.7733486e-18,0.0024658358,7.4548623e-25,1.991689e+28,-1.1627941e+10,-4.1151952e+34,-1.590461e-35,1.2268458e+32,-5.5002824e+28,-1.0164326e+35,5.6747445e+06,-3.4564563e+36,-17.327654,3.917033e+18,4.3795568e+08,9.6878474e-23,1.00258e-13,-4.734873,0.012941366,5.1493885e+19,9.218728e-21,-1.8201062e+23,-3.7309304e-19,0.00011094491,-5.992515e+31,-6.649461e+30,4.6746915e-36,-0.07755848,-1.2377087e-24,-3.5228738e+21,2.4929082e-12,-155.76761,-7.5559533e-28,-7.6216784,0.0025290253,-5.0977196e-17,3.888368e+11,-3.9635825e-19,-1.6234795e-12,-5.2270102e-12,6.538137e-09,-4.1245124e+37,121098.94,-4.2994082e+13,-2.6509525e+14,1.4483985e-35,-2.2675636e+13,4.784965e+12,-0.0007564223,-3.3444396e-19,913.97363,-1.0801325e+32,9.1071525e+34,1.9882515e+35,1.0287909e-26,-7.13333e-19,-8.0936885e-10,4.426515e+16,2.416185e-22,2.0876062e+18,8.209478e-33,1.4866014e-22,2.2370522e-11,-2.3724679e-20,-1.2949148e+38,-2.5730074e+09,-1.2521194e-32,1.2589636e-16,0.01983438,2.2584323e+19,-1924.9589,1.0456896e-32,-1937.7267,8.083348e+37,3.5634286e-12,-3.5141768e+24,7.6868525e+08,-1.00299e+26,-8.0853007e-25,6.2185344e-31,-8.661304e+17,6.029085e-12,1.18281834e-32,-2.0715544e-31,3.8691908e-38,6.848572e-37,-2.5675401e-26,9.6360424e-08,115.531425,-4.9488458e-09,2.0521947e-10,-2.1320873e-10,-1.3344061e-08,0.012999537,-7.608608e+22,-6.6494944e+28,7.89518e+25,-3.3257253e-09,3.84756e-34,2.4649715e+14,-7.9403933e+21,6.1912275e-38,1.8105269e+19,-1.1883992e+31,4.2187975e+33,1.7536294e+29,-2.2984794e-33,-2.4401034e-08,11.752845,-6.695194e-14,7.896003e-08,-1.2747712e+30,16260.711,-2.516914e+06,8.185241e-26,5.51349e-12,1.062586e+17,1.5837741e+27,-1.079151e+23,-227.33437,1.2370109e+14,-7.1148795e+22,-2.3331148e-18,-3.1011338e-18,-1.6325467e-17,5.849296e-15,8.313538e-37,2.2307462e-12,-1.1265689e-35,-1.0647719e+22,-1.0760589e-23,1.1089576e+11,-7.023046e+11,12.788189,3.3669271e+19,1473.9913,-4.2052702e+24,-0.1210549,1.4084291e-14,-2.2698856e-30,7.9373545e-21,-6.0225154e-34,13.690625,-6.166265e+10,3291.153,4.850365e+28,6.0251576e+25,3.2360094e+10,-4.1204842e-08,-1.8766237e-22,-5.940267e+24,0.12175986,-6.275708e+30,-3.5334542e+21,-2.3349988e-34,-1.1084991e-06,3.1602504e-10,6.1523744e-22,-5.12534e-16,2.4099915e+06,7.2030016e+08,2.5974713e+17,2.5907498e+18,2.7841578e-24,8.92164e+23,-1.8488735e+06,-1.0724923e+32,-8.3221306e+08,6.73472e+23,4.9082354e-18,-0.00015688047,-1.0311236e-34,-4.7817713e-17,-0.33760062,-3.7961178e+08,-5.5101903e-32,-2.3605965e-26,2.3730432e-33,1.8500722e-07,-4.2137997e+18,680856.06,-0.12130455,1.2389775e+11,-2.313723e+35,5.5514964e-28,-7.396549e-18,-1.0534758e-27,-14.242254,-2.0446392e-21,2.5560077e-21,-2.8631566e-25,-2.9018077e-20,-6.9263995e-32,1.4732186e+28,-4.5930205e-15,-8.5751623e+27,2.1572681e-24,-950.91785,9.836963e-28,-1.0596519e-23,-3.0764457e-24,-1.7280044e+15,9.029083e+08,-5.4385694e-08,3.2449426e+13,1.6347899e-18,3.4719342e-36,2.6433833e-11,-2.0775268e-15,4.1839007e-13,5.9816124e-08,387.14273,-2.5459175e+27,7.166225e-27,-1.77357e-16,0.8253731,-8.624703e+34,-0.000830683,-2.8539237e-14,-2.0176916e-20,-3.258637e+27,1.1192509e-10,1.6088755e+22,-1.4467139e-11,7.015338e+08,-4.7924583e-34,-1.2484353e-18,2.8462145e-30,9.28699e-28,-1.9702947e-21,-6.706319e-33,1.2542514e-27,-4.4950897e+11,1.5548152e+06,0.0017729944,1.4332271e+37,-3.990114e-24,2.0724328e-35,5.679913e-22,-1.1675225e+32,-5.502976e+32,1.9962696e-11,9.925662e-27,-3.1852536e+10,-1.6318371e+36,1.3582948,7.565862e-36,1.5552509e+07,7.3223084e+37,3.3013412e+38,-6.029273e-23,-6.837176e+21,-6.769021e+36,-2.5797153e-35,-1.6581943e+11,1.4879307e-37,6.1391654,5.82329e-17,-3.0332332e-30,-5.359805e+37,-3.358571e-38,-1.5440415e-24,1.0707602e-35,2.004862e-33,4.4643404e+15,4.6440476e-31,9.29308e+33,1.0449109e-05,-1.408172e+13,1.1084959e-09,-3.718982e-15,-94.79697,-2.1563587e-20,-4.5367677e-17,-1.4140469e-28,-1.1110845e+35,3.922702e+09,-2.0886348e-29,1.9112404e+28,1.3046028e-12,0.021669354,5.19854e+23,-2.0308408e+09,-7.717174e+37,0.32478693,-1.3212383e+32,1.7434062e-20,1.3640569e-05,2.2776722e+30,-1.1976334e+11,4.080764e-26,3.8431067e-24,-3.5863298e-08,2.120427e+31,3.2788357e-05,-177051.69,3.622023e+24,-1.5699886e-20,-6.0745604e+28,-3.246067e+35,2.7970018e+17,-0.0068757692,-2.9281849e+22,-1.7668866e-28,4.7594204e-07,3.7319674e+15,-1.0249999e+17,5.402366e-25,-1.4393297e+34,1.3583233e+31,-842.3952,254.39806,-0.00010679667,1.5877648e-21,-1.2204946e+36,5.1031017e+31,-3.685327e-16,-4.5568803e+35,1.06031e+37,2.4380624e+26,4.4611005e+33,6.4821096e+28,6.4456684e+13,-1.0107718e+14,9.311345e-26,-1.1143116e-27,6.2505643e-31,9.1283575e+25,-5.7962483e+24,542.84326,-4.075951e-34,-1.3241308e-10,0.00083800225,-1.0724585e+22,-1.0749937e-27,6.60706e+28,3.7044731e-22,4.540836e+27,2.394899e+07,-5.826946e-34,1.0884123e+21,-6.2083053e+32,2.1197162e+36,5.873944e-29,0.9339197,1.6475825e+14,8.1153506e+34,-4.116348e+08,-6.7253846e+34,-2.3705526e-33,3.363346e-22,-8.621884e-36,-1.9778576e-32,-5.645728e+31,-2.9538522e-38,-5.0888503e-14,-5.9284887e-34,-2.4281877e+30,1.6529641e-38,9.2524464e+24,-5.4598033e+34,-8174.8286,-2.4323371e-29,-6.7514793e-21,-0.0029436466,6.931379e+36,-9.710433e+27,-5.7779515e+20,-6.0160737e+34,5.1097145e+15,5.6236387e-11,3.3391025e+06,-1.401638e+24,1.8176333e-20,21.807034,-2.6888694e+09,2.1493768e-10,1.1882333e-27,8.533532e+31,-5.0750655e+37,7.8921426e+34,-1.5049355e+30,6.7334703e+33,-5.43746e-13,1.560606e-16,4.1685565e+32,2.405715e+28,-8.280288e-34,4.996571e-13,-1.0657595e-15,-5.7180725e-18,2.792794e-20,-2.7687032e+26,-4.436979e+14,1.7023573e-10,-5.2386927e-12,-0.11775814,-43703.793,-1.2891121e+24,0.19201943,-2.8314747e-16,-1.8507362e-31,-1.0501605e+21,1.0537694e-22,-2.8007195e-12,1.8478246e+08,-0.00078039867,3.4921868,-1.2048215e+24,3.2674637e+11,1.338547e+15,-1.1212691e+27,1.1422243e+36,1.3501801e-30,-3.2354026e-18,9.94502e+37,78.18397,5.738127e-26,-244.36287,1.0546671e+08,4.4296292e-29,-4.283524e+24,-4.967636e-27,-3.5063696e+28,0.002785674,-2.0783573e-11,1.2525857e-09,-1.5446001e+13,2.7854849e-11,3.1445646e+24,-1.2969352e+36,2.1269896e+23,-1.6582518e+24,2.289771e-19,-4.8245055e+33,2.2805093e-23,6.5541813e+13,2.5180647e-26,2.4414836e-36,1.4315112e-05,1.0089414e+23,2.0356595e-11,7.836733e-21,-1.1031004e+31,-1.00312325e-36,5.673007e-22,-1.4186337e-19,-58.19817,-5.2318764e+20,1.7442418e+22,1.1708117e-09,-1.2338359e-27,2.6827807e+34,-2.2518135e-36,3.4113605e-06,-5.1884316e+21,7.4716306e-37,1.699572e+19,-5.476133e-26,14.5582485,-7.364205e-10,1.6463716e+11,8.4886905e-23,-1.3815333e-05,-7.058309e+14,1.0452907e-36,-1.4659122e-18,-8.019365e-37,-7.260705e+33,2.1076e+20,-4.4857703e+20,3.0911382e+22,1.3009763e-15,-6.872864e+33,14084.606,-1.3054075e+26,1.4953374e+22,5.2683224e-29,-11367.043,-2.1255378e-32,84.90825,1.2485981e-08,5.0762103e-24,-1.1607827e+07,7.003939e-27,-4.383733e-32,5.307492e-26,-1.5888435e-33,-8.733494e-07,-1.1444419e-11,2.672764e-09,1.0853306e-06,-5.0064485e-24,-1.9629876e+30,-4.403385e+35,9.640736e+33,-6.4588e-11,-3.3004912e-12,1.1814637e-10,1.8540686e+28,3.9162243e+31,-4.6006054e+22,1.26019545e-20,2.2463284e+17,1.6502826e+17,2.1039641e+21,-7.227404e+23,2.524884e-31,-9.67103e+16,2.9287998e+32,4.2890275e-12,9.1799116e-08,-3.479065e-33,-1.9355209e+09,0.40564343,-3.1402504e-29,9.602471e-08,-3.119519e-23,-9.68504e-37,-4.837734e+25,1.1984106e+09,175203.86,-1842.4025,-1.6700136e+18,7.948736e+22,-1.1140965e-09,2.720159e+13,-1.5381953e-20,5.9568394e+15,1.9751082e+35,-7.926566e+25,7.713685e-16,-538.8496,-2.5496981e-38,6.3380864e-31,1.8592308e-25,-2.7615418e+15,-1.5912473e+25,5.8309456e-34,-8.0305406e+29,2.7127472e+33,-8.930654e-39,2.2439675e-05,1.1730617e+21,-2.92941e-32,2.107157e+20,-2.3357908e-14,-2.2338921e+15,1.0654349e-18,1.1787299e+37,22.612043,-0.00093492604,45.423462,1.2718226e-35,-2.6409508e-34,5.0076683e-16,-4.4825538e-33,-1.756044e+33,-4.2333902e+12,4.4376142e-17,3.022611e+25,-3.94513e-17,-2.0701536e+10,6.2540356e+11,2.6780685e+11,-5.604448e-33,-2.1685264e-26,-1.2099394e-29,-6.3943153e-09,-7.980684e-33,-6.821687e-22,-5.032411e-33,-1.4237913e+31,2.8022016e+09,-1.3813125e-28,-2.6925276e+31,-4.476906e+26,1123.05]':::VECTOR,19:::OID,'-40 years -10 mons -602 days -14:38:01.552088':::INTERVAL,869:::OID),('[1.1658961e-17,6.463529e+30,1.0912715e-24,-3.384548e-31,0.9126112,3.080908e+29,2.270395e+31,-2.8122706e-16,-0.00018538583,2.420663e+23,9.884821e-15,3.369772e-11,1.428303e-20,2.349702e+26,-4.1750222e-35,-4.276944e+08,-7.0793895e-35,-1.08156155e-32,0.07423122,1442.6783,1.4468222e-30,-1.9084657e+14,-4.9171107e+22,2.9501715e+32,5.463476e+16,0.0045625544,4.1480002e-24,-4.6512107e+14,2.4669225e-22,9.364332e-27,-3.881258e-08,-2.5537405e+17,-17.424818,-1.5480028e+26,7.077175e-28,8.830448e+23,3.1689392e+06,-10.85476,-2.1046968e+23,-1.7418142e+31,7.263211e+27,-5.933134e-06,1.7202808e+18,1.4391294e+31,-2.8587868e+06,7.333432e+37,7.411575e-30,-1.08640284e-10,2.1218295e-21,-4.3536277e-21,-2.817914e-12,-3.7798879e+09,-8.938817e+32,-1.0315184e-26,1.8833338e+10,1.302276,4.5491853e-25,1.2858783e-35,-5.240265e+25,2.085428e+35,-1.6295895e+12,-5.336334e+08,-1.57919e-12,3.0426513e-24,-7.846816e+28,2.2989469e-23,-5.4697905e+28,-6.824198e+24,-1.4118512e-21,-5.8505228e+07,1.36377795e+14,-3.991357e+37,0.0011439164,838830.5,-5.6325894e+20,3.91406e+06,-1.7809858e+16,4.7117093e+17,-7601.762,-4.823603e-06,-406846.28,-5.910431e+06,-600975.5,32.106873,-3.4501995e+19,-4.7166454e-06,5.125644e-28,9.146988e-05,-1.843813e-12,1.7046247e-14,1.3589838e-06,0.046407208,-1.8123829e+29,-1.1531832e+35,1.3748645e+15,0.0013367372,-6.4913006e-12,3.006138e-32,478.05344,6.3877467e-24,1.0671408e+15,-1.3967648e-38,8.575836e+11,-2.91186e-23,-166.42953,-1.0853632e-20,-6.027653e+27,-4.2251557e+25,1.397878e-25,-1469.0051,-3.2005252e-24,1.4373707e-23,7.3319953e-37,-0.07243696,-1.7668378e+07,2.3812532e-18,1.2816323e+14,-6.4885757e+30,-1.3612592e-28,-16134.195,-9.758204e-21,-9.614333e-06,2.7943199e+34,9.549343e-30,2.5436202e+20,1.4667952e-29,0.0002618875,4.116559e+08,7.162827e-09,-1.2240625e+08,8.763925e-21,1.129567e+09,2.6739368e+06,-4.1911543e+31,-2.4861574e+25,2.709377e+18,8.1780623e-16,-1.7746629e+15,-1.2278448e-34,7.424799e-13,2.2994836e+21,-20.569662,-27715.418,-1.1043004e-26,-2.8224837e+26,1.5068381e+27,-1.3414186e-32,1.22576994e-23,5.735997e+21,-3.413784e+23,5.5868713e+26,-1.2733835e-06,-4.345986e-33,1.5517224e+29,-3.4000597e-37,-1.197103e-10,0.00012634201,1.252295e-24,6.0266253e-10,-9.937685e+11,5.3201204e+35,5.2574937e-25,0.13838078,-2.534381e+18,1.0081061e-22,1.0431098e+14,-0.10133913,-4.10265e+33,1.6080376e-27,-9.973752e+06,-1.1980332e-33,-4.4914e+22,-1.0754168e+06,-2.2363937e+14,-5.605698e+23,7.252492e+08,1.6310396e+15,-3.886644e-29,8.197231e-26,1.02397766e+33,1.8552083e-17,0.022222634,-3.7650585e+16,3.144028e-34,-3.222565e+14,-3.847509e+21,1.0202449e-38,9.823126e-19,-3.8381523e-07,-2.3125783e-24,3.703277e+29,1.1443741e+38,1.2771876e+20,5.523603e-28,-3.9242973e+11,-3.4383828e-31,-7.197054e-29,-1.00857276e-32,8132.6543,0.038889874,1.645575e-25,-9.375132e-13,-4.0946378e+26,6.9721023e+22,8.529881e+16,8.0768364e+36,-9.233866e+29,2.2820054e-20,-1.9556152e+37,-1.840736e-31,2.0620527e+30,0.26149663,-1.320978e+37,-1.3382682,2.7555371e+17,7.292669e+15,9.286016e+16,-1.7018585e+06,1.2467001e-10,3.7719768e-32,3.288659e-31,2.9368737e-20,8.229473e-21,-7.2752075e-13,2.281144e+26,-6.6592073e-16,1.2633875e+34,2.4935759e+32,6.6634086e+08,2.7046175e-24,1.3409843e-22,1.3806999e-05,-1.0966508e+26,3.160451e+15,2.0205466e-10,2.359685e-07,-6.652215e+26,-1.2270278e+12,-4.957845e+14,1.3585777e+38,8.855922e+32,2.0903675e-08,4.552146e+36,12.675517,2.3295327e+16,1.2771474e+23,1.7584327e-16,-1.4310526e+12,-6.4139423e+09,2.4948236e-31,4.0315628e+06,-2.566385e-29,-5.9565948e-05,9.121214e+37,-1.0635666e+25,8.9148815e+36,-2.113762e-16,-1209.9382,-1.0120064e+24,-2.1892759e-36,691646.7,4.0347797e-10,-3.0416405e+06,-7.929576e+08,3.0152908e-24,177.6664,6.881979e+37,-0.00091879483,2.8940505e-05,1.1964733e+07,4.883741e-27,-3.465497e+31,-5.9707963e-09,4.8639453e+29,-5.7831326e-24,8.786313e-17,1642.8318,1.0347676e+35,-1.4790213e+08,-2.1516383e-08,2.4372018e-09,-3.3397597e-21,-2.1183334e+37,6.216937e+18,-9016.95,2.909891e-11,1.4714322e-16,3.1129154e+30,1.5474366e-24,-1.6435906e-36,-4.0009055e-31,-4.677993e+12,4.6893162e+36,3.3675043e-37,1.4644424e-18,-2.2168319e+17,1.6004686e-13,0.0004839631,2.1853773e+14,-8.5966056e+07,0.0007086807,2.0752692e+07,1.2202468e-16,-4.4086467e-11,-1.1735538e+14,8.741022e-37,-4005.0613,-2.7135565e+16,-1.4909754e-37,-2.6310396e-33,1.0151183e+09,-3.7885693e-28,5.765925e+17,-2.8107148e-09,-0.00069966586,1.3728721e-10,-1.1137128e-10,1.8603504e-29,599.3762,5.8692805e-34,-3.478798e+34,-3.7936593e-07,-5.371228e-17,70.37169,-0.00099858,-4.457669e+14,2.8442153e-33,3.3918312e+17,7.8207417e-28,1.1018284e+35,-2.5604543e-29,1.3194986e+32,2.5363063e+10,0.00019085697,1.0093948e-17,-2.8416795e-34,7425.1636,-3.718743e+29,7.1129734e-36,1.190099e+36,-1.4228203e-13,7.337205e-06,1.1366812,30.991484,-78.691055,-1.8041798e+34,6.8805513e-22,5.831703e-11,3.2386228e+27,-4.370663e+07,-4.737971e+20,3.0489712e-17,1.2792003e-26,5.2829287e+11,9.0499814e-13,2.9242272e-05,1.0556828e+17,-45871.754,7.812632e+10,-2.1069785e-29,2.1841936e-16,-1.1247697e+33,1.2154636e-10,-16770.783,8.5397625e-32,1.5488378e+38,-4.310939e+21,1.0248031e-10,3.0289663e-11,-2.407824e-07,-4.7194976e+07,2.2754698e-32,2.931414e+15,-2.1307989e+25,6.289226e+07,4.0430745e+19,6.7473e+11,0.012456237,-2.450551e-14,4.2186212e+23,1.975509e-36,-1.2365412e-15,-2.7927763e-28]':::VECTOR,19:::OID,'7 years 3 mons 890 days 12:45:04.491225':::INTERVAL,1560:::OID),('[-2.4661966e-19,1.6702357e+13,729853.4,-2.0313724e+32,-2.3807502e+27,1.7182714e-27,-9.451804e+12,1.6702047e-06,2.2937569e+13,-1.2232283e-35,2374.036,-4.9855802e+30,4.2533055e+06,1.2173888e+27,3.4384205e-19,-4.3030966e+08,-2.326e-34,6.2103272e-06,3.7146295e+23,6.1423484e-21,1.7894486e+23,-2.8218146e-12,-2.2504808e-21,-2.3044731e+17,-300788.66,6.4146676e+28,1.9860315e-13,28615.482,-0.00067334686,-8.517175e-11,1.0642495e-25,-4.3188702e-29,-2.1261836e+16,-2.0690624e-32,-1.8359834e+28,6.3039696e-37,-7.889989e+25,-5.06294e-12,1.499874e-08,-5.6755614,-2.0389261e+36,3.02329e+18,1.7820481e-27,4.940327e-09,-4.234397e+11,-1.28051605e+23,-1.3168838e+09,2.7865584e+28,1.1226925e+10,-2.0793769e-29,3.0470063e-10,-9.667018e+07,7.843485e-05,8.359142,7.7601664e-28,4.872026e+27,9.291286e-36,4.1988216e-38,-1.9832844e+31,-1.6274859e-16,-3.3489383e+10,5.207295e-05,2.9612199e-08,8.248159e+19,-3231.1294,103.98256,7.386031e+22,1.2460517e-35,2.4842692e-26,-5.700153,6.162989e-14,4.9231043,4.654187e+32,8.72036,8.315183e-12,7.377164e-26,-2.0105681e-11,-9.285536e-25,-1.8014399e-14,-2.515188e+33,2.888146e-38,4.530913e-11,-2.746448e+32,2.7548745e-06,3.8516964e+18,-8.058252e-19,-12.387784,-1.2743148e-38,8.8640376e+11,3.38852e-29,2.326835e-23,8.820834e+28,2.8986263e-21,-5.557635e+22,-2.7553384e-14,-9.5396974e+27,-3.896334e-21,-0.5168553,204.84592,-2.4969173e-33,-4.5910146e+15,2.4177415e-11,-3.6019586e-17,-2.0317784e+27,-1.6463351e-24,-1.13303314e-32,3.5392968e+22,-2.0032715e+08,-1.871477e+18,-7.73669e+08,6.9497667e+09,1.241725e-11,4.090253e-18,4.184287e+20,-1.225915e+26,-7.2254534e-07,1.5109206,-2.332197e-36,-5.3722863e-38,3.368241e-10,1.9006272e-08,-2.9626507e+31,1.6926037e+14,4.205205e+26,2.5565812e-11,-5.6895824e-13,1.3889949e-21,4.3471228e-35,6.9223944e+31,6.786473e-37,4.9799465e+33,-3.253329e-05,-3.5364206e+21,-6.081386e-08,-2.0276214e-07,1.3494129e-21,8.554353e+19,4.5730817e-34,1.5369185e+17,1.04825995e-19,-5.284926e+24,-4.2772615e-19,-1.2266878e-08,2.7683914e+08,1.6293584e-12,5.6178812e+16,-0.742406,-7.9075456e+24,-6917.528,-8.919232e-35,-4.486471e+21,-8.235269e-32,5.1715963e-23,-3.297909e-16,-2.4938395e+28,-2.4505838e+13,15240.708,1.11090355e+23,-2.5279605e-34,-1.5058969e+37,5.394178e+34,6.3156043e-12,1.5099823e+23,-8.385659e+32,-7.777633e+34,1232.6146,-2.3103332e+17,1.3266023e-20,3.4095163e-10,-21.657166,1.7869404e+37,6.1068315e+13,1.6802784e-09,1.2599296e-19,890198.94,1.7824961e-24,7.2383325e+35,-3.0057772e+12,3.6170913e-24,1.0585629e-11,1.3846722e-31,9.388231e-21,-2.0845175e+18,407.6722,-7.828813e+23,3.005716e+08,545.8056,2.7465093e-18,-5.1817383e+37,-3.9939995e-19,2.286034e-05,-3.996281e+31,0.0553659,-3.433755e+34,-1.3975395e-33,-2.8847987e-30,2.4023134e-33,2.2296622e+13,8.637493e+24,-3.5085982e+18,28613.703,-1.874978e+19,-2.0923371e+15,105058.2,1.2315625e-19,3.5848185e+17,-8.940776e-26,-4.8646818e-20,-4.2027562e+11,4.6534718e-10,3.654636e+32,-3.7832502e+28,6.5200184e+09,-2.285075e+29,-3.8603378e-16,5.1379642e+25,-4.396823e+29,3.704246e-25,-2.2014989e-34,-4.1392218e-07,0.0018567921,4.8732855e+11,8.419361e+26,3.796214e+20,3.126246e+15,1.295687e+20,2.5318427e-37,2.0633215e-21,-4.4091716e+10,3.3242749e-18,-5.146929e-31,-1.3663571e-14,-1.7224547e+37,3.8328866e-28,191.12134,1.1015491e+35,-3.63665e+24,3.8194025,-9.8039456e+08,12.102576,-3.881597e-26,-1.293909e+38,-6.9552487e-31,-7.852157e+29,6.840881e+07,2.2073142e+14,5.3778243e-37,-0.09924895,-13487.456,4.7126886e-35,2.5730593e-15,5.0957063e+32,-3.514817e-08,-421.05054,1.941668e-10,4.7970417e-20,3.8593682e-38,-4.799863e+37,59268.223,-1.8277545e-28,-8.368391e+14,-1.8371015e+15,-10343.978,-229.76572,-3.445384e-31,-1.4909897e-33,-2.4666393,-5.2265987e-12,-1.17866996e+30,-1.5218575e-13,2.5242776e+34,-1.7864147e-08,-2.3859091e-33,-3.9694142e+30,5.87131e-26,3.9928932e+07,-1.392591e+26,3.314987e+08,1.13467725e-20,1.3728983e-24,-1.2687542e-19,3.6773767e+19,-1.5559601e-18,-5.5614128e-05,3.0058466e-38,-360121.6,1.0497698e-05,-3.3501007e+25,2.0275217e-29,1.303325e+24,-4.6161307e+27,3.111465e+34,9.229169e+13,4.8378946e-28,-7.7002404e+12,-4.8198392e+11,1.434768e-23,-7.880811e-05,9.742817e+24,-3.7015034e-37,-5.244442e-38,7.031896e+23,-0.00041080912,8.5418664e+15,-4.6653064e-14,3.206408e-08,4.602163e-32,22.62088,-3.00858e-21,-1.9208352e-11,8.360942e+06,1.3268719e-19,-3.43577e-13,3.8084687e-28,6.0566772e-18,-9.437219e-13,1.1846439e+07,-5.2412165e+13,-1.2862907e-17,-3.3869702e-23,-8.98444e-29,-2.004255e-15,-1.8122343e-37,-6.6272206,-2.2779847e-27,-1.2139547e+14,-1.3552545e+34,-3.1227557e-32,7.874078e+36,3.8304934e+23,9.835372e-29,7.482308e-12,-1.01383995e+31,3.5210202e-37,9.2102895e-36,4.3999986e+10,-1.3618559e-10,1.16923854e-29,6.324489e-06,2.6636946e+09,5.7149334e+21,0.0034446216,-1.3251508e-23,-1.2659203e-09,5.767274e+26,-1.5559041e+15,2.5873082e-37,-3.7206962e+31,-1.2912246e-13,-7.978252e-24,2.0158568e-36,4.0410125e-08,4.518498e-29,2.7259883e+22,2.1795387e-26,2.2104288e+20,-5.507555e+26,-2.6131176e+24,1.8343532e-28,694.40247,2.58994e-10,-1.6244153e-31,6.5947707e-15,2.2068642e+06,122.94401,-2.1705425e-24,-4.631769e+36,-6.32234e+23,-3.4414736e+07,-6.4346525e-05,7.837841e-26,2.8724026e-38,-1.4116221e-38,-1.2503988e+27,-3.8431787e+34,6.740732e-12,-1.2576497e+11,2.998391e+20,4.2132504e+20,-2.1929033e+22,-1.6061348e-07,1.7954604e-12,1.6784129e-15,-4.271443e+19,165.40019,-5.812177e-24,-1.0559573e-35,3.5765403e-15,-3.938754e-29,74933.43,1.2213013e+17,3.3472934e+22,6.1214755e-10,3.0962661e+12,-1.521325e-14,-6.340373e+28,-2.2011048e+17,-4.009324e-12,9.7723324e-32,1.172784e-10,-2.777314e+15,3.4088611e-38,-7.946604e-27,7.3102047e-38,-4.472851e+37,1.8889061e-19,-1.690084e+06,3.1944206e+19,7.380758e-34,-8231.576,6.4743585e+21,-1.7851543e+15,1.4040922e-28,1.710919e-34,-1.2275815e-20,0.37829018,-0.82898945,7.108426e+25,1.014164e+15,1.5509052e+38,3168.8298,7.091482e-20,-1.347227e+21,-1.10821075e-11,-0.0017004084,3.294852e-16,-1.7177975e-08,1.0591107e-30,-3.260299e+31,-1.8511161e+30,-3.3404938e+10,2.6369707e-11,2.6492667e+32,-1.8176783e+28,6.570297e+35,6.94723e+37,1.5737225e-28,1.5665853e-09,1.8872241e-10,9.600047e-07,4.5455677e+08,1.3636526e+22,-2.0784445e-11,3.0862019e+10,4.3569787e-11,-3.2623895e-10,-1.5784165e-25,2.9262735e+36,-1.1566181e-26,-0.0130745545,9.662587e-11,7.5861247e+37,2.855248e+11,-3.1465705e+19,-9.30347e+27,-8.302373e+25,-2.390041e-30,3.4056755e+33,2.579719e-35,-72.846085,4.1488092e-32,6.296964e+35,14301.715,-4.4655563e+36,-5.9250904e+16,-19.627718,31.56007,-9.442351e-05,1.4398298e+15,-2.9166133e+14,1.079531e+28,-2.1358763e-12,3.2845508e+17,3.8277238e-38,7.9122833e-25,3.565692e-17,8.0308596e+16,5.9600233e-27,2.9132747e-30,6.0868446e-35,4.7076275e-05,-5.772563e+33,1.3004909e+32,-5.4339747e+28,-0.007906328,1.8450562e-37,7.591791e-19,218.87845,1.5960286e+24,3.7269066e+09,-5.4532193e+17,-1.089807e-06,-9.560816e+15,-8.215893e-12,1.2716947e+26,-2.3016962e+29,-2.5760195e+28,2.112916e-11,-8.761674e+17,1.3681697e-35,0.4765893,-1.3252968e-11,8.885892e+07,-42179.438,-1.9931152e+06,4.5276056e+07,-5.459209e-30,-1.9312922e-20,-2.1070001e-25,6.5240626e-25,-1.3661489e+11,7.070177e-35,1.597155e+29,-2.6012743e-38,3.0419721e-25,1.129657,-4.370447e+18,-1.8699403e-07,-2.3110653e-07,1.0601622e+34,-2.9623776e+32,-8.9212335e-07,7.33347e-06,-0.00042502256,-0.0033264244,-0.020620454,-1.1891449e+22,-436.47614,445.73477,-7.025076e-05,3.774121e-20,-8.767435e-23,3.9656229e-07,-2.426356e+26,1.5889703,9.946981e+11,6.4578133e-38,-6.357911e+15,-7.4636985e+34,-8.1233623e+09,5.5457792e+34,-1.8245042e+29,1.3431881e+25,-2.1067628e-05,4.0300253e-36,-7.6960735e-22,7.107371e+07,-6.5409196e-32,6.318622e-12,3.696859e-29,2.9196373e-25,1.5310722e-07,-1.258531e+09,-1.74695e+08,1.0858805e+28,-3.7789605e-23,1.5288365e-16,-7.800791e+17,-7.998758e-33,4.4514558e-26,99797.555,-197.52185,-3.01441e-07,-3.4288508e+06,-2.9933975e+23,1.77788e-05,-1.2447135e+07,8.4519104e-38,-4.086646e+19,-1047.8201,-2.2099573e+24,2.71073e+23,-8.735656e-10,-3.11246e-26,1.6762589e+28,5.7873494e-16,-1.773285e-07,-1.3585184e-31,-1.5970015e+32,-2.0323272e-30,-1.9904708e+37,1.8188598e-38,-1.5289981e-34,-510720.8,1.9254692e-28,1.2395439e-25,-1.1932165e+06,0.044392157,-0.2041046,5.2812454e-34,-7.298422e-13,2.0026445e+08,-1.0262755e+11,1.3125441,746.9293,-4.352225e-38,1.1453868e+27,-2.3540769e+25,-1.27938615e-11,1.6079624e+37,2.5123222e+37,1.2878474,-2.1909724e+07,-7.4287076e-32,-3.4666446e-06,-1.4008628e-17,2.574546e+16,-4.4274758e-23,-3.9461637e-17,-4.721118e+36,1.8051082e+06,3.649651e-17,-1.29802565e-23,-1.0778957e-28,2.757869e-20,-4.9277128e+32,4.219627e+18,9.68661e-18,2.252075e-37,6.6454127e+19,-7.220347e+06,5.466363e+23,-7.629739e-32,3.8374958e+25,2.6455515e+33,-2.362325e-37,6.2157833e+35,8.276365e-31,-1.7072006e-33,-860.06256,-8.8453987e+15,-5.887582e-08,8.3735547e+15,-7.663065e+11,-7.29179e+20,-3.1476836e+34,2.1958192e+15,1.4536188e-14,-1.3528721e+14,2.1861474e+30,-0.0009177288,-0.006758344,6.298488e-39,-2.3479158e-11,-3.5984034e-08,-1.0397013e-11,-1.1231108e+32,-2.8695342e-32,-3.6516276e+22,-2.168204e-27,2.3460024e-07,-3.8916058e+06,-2.959927e-37,-7.310858e-23,3.10684e+33,1.4302498e+29,8.117367e+30,9.045188e+22,-0.12994833,5.085223e+24,-5.140421e-18,-5.298562e-07,1.8279627e+20,2.3544188e+23,1.7628208e-17,-7.7839917e+34,8.472798e+34,4.6429056e-15,9.558245e-30,4.8116915e-31,-5.3593096e+10,2.068985e+30,0.29711035,7.15073e-35,-3.2735777e-14,-63.351902,4.035054e-15,1.4558056e+20,1.0459718,-3.9502773e-19,1.0099262e-06,-1.03170775e+17,1.769725e-28,2.7065735e+29,-0.00040483306,-7.2920394e+37,-5.638242e-20,2.6491192e+27,-3.220101e-07,-2.7795316e-19,-5.614945e-06,1.8227462e-19,5.27586e-35,-1.5094423e-19,1.6722724e-06,-0.05829514,-6.594102e+12,-4.1287048e-26,-8.005138e+09,-9.1964305e+30,-1.3098082e-21,-2.5581941e+19,2.1677187e+28,-2.2850183e+27,-7.526218e-05,7.55987e+33,4.3905408e+12,-2.3551278e-12,-8.8514406e-07,4.4837374e-21,-52802.688,-1.8450893,-2.3350776e-08,3.6067833e-09,-5.184511e-32,3.3541873e-20,4.7433883e+37,-6.38603e+17,-0.00010409461,-1.7506344e+18,2.3381446e+38,1.5070215e+30,2.3192864e+28,2.508035e-37,-2.597689e+12,1.4545431e-29,2.503011e-19,-3.536601e+18,-6.488667e+22,-1.9722888e+21,1.0898368e-07,6.87984e+27,9.399008e-15,-3.998204e-38,-0.0014182781,4.295651e+11,1.0740023e+35,1.6322907e-16,9.7012534e+26,3.904305e-31,-6.6092047e+31,-1.3415101e+09,-1.6229754e-07,-7.360894e-05,6.773367e-19,7.0176e-41,-2574.776,1.3891353e-30,2.7682571e+20,0.0007998075,5.3819145e+20,-5.334605e+33,-6.445283e+19,0.0012615513,7.945384e+32,-5.1827696e-20,-2.575333e-30,0.006657507,-1.1559595e+25,5.556566e+13,1.2528554e-16,1.2159743e-17,1.7078452e-22,4.0554978e+20,-1.7568283e-13,-9.1719133e+21,2.0135577e-21,5.713073e+12,2.4202863e-17,-9.797477e+13,2.8109305e+06,1.5982167e-22,-2.9849914e+07,4.7545882e+36,-1.2640165e-33,1.2294948e+37,1.7742413e-19,1.2266296e+37,-1.10138926e-10,-2.7151555e+24,-6.381513e-11,-6.73104e-28,-6.0168647e-16,4.232772e-20,-2.4781206e-21,-5.9585987e-31,4.4261597e-07,3.282571e-21,2.5810066e-18,-1.9733626e-22,9.952944e-38,-0.008491607,1.1654652e-24,-6.491775e-08,-4.9661543e+14,-1.570705e+30,-1.94815e-10,-1.8134292e+35,-5.511971e+17,1.0194474e+08,-2.1572556e+11,-27.362864,1.346607e+32,1.085076e-36,1.914333e-27,9.793324e-09,4.35144e-20,3.0335539e-12,1.0362479e+06]':::VECTOR,1186:::OID,'-61 years -6 mons -498 days -20:05:24.34473':::INTERVAL,2206:::OID)"
  ],
  "usesDeclarativeSchemaChanger": true
 }
}
    schema_change_external_test.go:134: 
        	Error Trace:	pkg/ccl/testccl/workload/schemachange/schemachange_test_test/pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:134
        	Error:      	Received unexpected error:
        	            	***UNEXPECTED ERROR; Received an unexpected execution error.: ERROR: command is too large: 144693409 bytes (max: 67108864) (SQLSTATE XXUUU)
        	            	(1) forced error mark
        	            	  | "fatal error when running txn"
        	            	  | github.com/cockroachdb/errors/withstack/*withstack.withStack::
        	            	Wraps: (2)
        	            	Wraps: (3) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*opStmt).executeStmt
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/operation_generator.go:3178
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).runInTxn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/schemachange.go:499
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).run
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/schemachange.go:587
        	            	  | pkg/ccl/testccl/workload/schemachange/schemachange_test_test.TestWorkload.TestWorkload.func4.func6
        	            	  | 	pkg/ccl/testccl/workload/schemachange/schemachange_test_test/pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:123
        	            	  | golang.org/x/sync/errgroup.(*Group).Go.func1
        	            	  | 	golang.org/x/sync/errgroup/external/org_golang_x_sync/errgroup/errgroup.go:78
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_arm64.s:1222
        	            	Wraps: (4) ***UNEXPECTED ERROR; Received an unexpected execution error.
        	            	Wraps: (5) ERROR: command is too large: 144693409 bytes (max: 67108864) (SQLSTATE XXUUU)
        	            	Error types: (1) *markers.withMark (2) *schemachange.ErrorState (3) *withstack.withStack (4) *errutil.withPrefix (5) *pgconn.PgError
        	Test:       	TestWorkload
Schema Workload Stats
Total Schema Statements Executed = 71
Total Schema Statements Succeeded = 71
Total Schema Statement Expected Failures = 0
Total Transactions Committed = 38
Total Transactions Rolled Back = 30
Total Transactions Executed = 68
    schema_change_external_test.go:99: backup, tracing data, and system table dumps in /artifacts/tmp/_tmp/67537770b84c3a4ea177539c1e46bd43/logTestWorkload1432192498
    panic.go:626: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/67537770b84c3a4ea177539c1e46bd43/logTestWorkload1432192498
--- FAIL: TestWorkload (9.32s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #131002 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot T-sql-foundations branch-release-24.1.5-rc]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestWorkload.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

pkg/ccl/testccl/workload/schemachange/schemachange_test.TestWorkload [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17123853?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17123853?buildTab=artifacts#/) on master @ [6941bc20208800b7c1b7e81d3a26517b780de7f4](https://github.com/cockroachdb/cockroach/commits/6941bc20208800b7c1b7e81d3a26517b780de7f4):


```
     [
      "public",
      "table_w0_29",
      "col29_w0_39",
      "col29_w0_37"
     ]
    ]
   },
   {
    "query": "SELECT (false = ANY (ARRAY[false = false, true = false])) AND (count(*) = 0) FROM public.table_w0_29 WHERE col29_w0_39 = (false)",
    "result": true
   }
  ],
  "previousStatements": [
   "INSERT INTO public.table_w0_29 (c̠ol29_w0_30, col29_w0_31, \"co\rl29_w0_32\", col29_w0_33, col29_w0_34, 😳col29_w0_35, \"co\"\"l29_w0_36\", col29_w0_37, col29_w0_38, col29_w0_39, col_w0_29_w1_90) VALUES ((-2.896320953434952181E+22):::DECIMAL,e'\\U0002E174\\U000163F1\\U00080E77' COLLATE en_US,'BiCW5%uI'::NAME:::NAME,'05:30:29.091723':::TIME,1082:::OID,'w'::BPCHAR,'\"a\"':::JSONB,false,'BOX(0.39625284055582366 -0.35653310818765427,1.5418699919066352 1.562225961921797)':::BOX2D,false,'1971-08-16':::DATE),((-574729007196.5333189):::DECIMAL,'' COLLATE en_US,''::NAME:::NAME,'18:24:18.89204':::TIME,19:::OID,'\u0026'::BPCHAR,'[\"baz\"]':::JSONB,true,'BOX(0.14968352545770863 -0.7893949453210714,0.7554578560676228 -0.17215925625126083)':::BOX2D,true,'-infinity':::DATE)"
  ]
 }
}
    schema_change_external_test.go:134: 
        	Error Trace:	pkg/ccl/testccl/workload/schemachange/schemachange_test_test/pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:134
        	Error:      	Received unexpected error:
        	            	***FAIL; Failed to receive a commit error when at least one commit error was expected
        	            	(1) forced error mark
        	            	  | "fatal error when running txn"
        	            	  | github.com/cockroachdb/errors/withstack/*withstack.withStack::
        	            	Wraps: (2)
        	            	Wraps: (3) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).run
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/schemachange.go:672
        	            	  | pkg/ccl/testccl/workload/schemachange/schemachange_test_test.TestWorkload.TestWorkload.func4.func6
        	            	  | 	pkg/ccl/testccl/workload/schemachange/schemachange_test_test/pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:123
        	            	  | golang.org/x/sync/errgroup.(*Group).Go.func1
        	            	  | 	golang.org/x/sync/errgroup/external/org_golang_x_sync/errgroup/errgroup.go:78
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_arm64.s:1222
        	            	Wraps: (4) ***FAIL; Failed to receive a commit error when at least one commit error was expected
        	            	Error types: (1) *markers.withMark (2) *schemachange.ErrorState (3) *withstack.withStack (4) *errutil.leafError
        	Test:       	TestWorkload
Schema Workload Stats
Total Schema Statements Executed = 140
Total Schema Statements Succeeded = 140
Total Schema Statement Expected Failures = 0
Total Transactions Committed = 83
Total Transactions Rolled Back = 72
Total Transactions Executed = 155
    schema_change_external_test.go:99: backup, tracing data, and system table dumps in /artifacts/tmp/_tmp/67537770b84c3a4ea177539c1e46bd43/logTestWorkload1770852805
    panic.go:626: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/67537770b84c3a4ea177539c1e46bd43/logTestWorkload1770852805
--- FAIL: TestWorkload (13.88s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #131002 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot T-sql-foundations branch-release-24.1.5-rc]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestWorkload.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

pkg/ccl/testccl/workload/schemachange/schemachange_test.TestWorkload [failed](https://tanzanite.cluster.engflow.com/invocations/default/31143ae0-8241-4777-89dc-db9fca8d1632?testReportRun=21&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL3Rlc3RjY2wvd29ya2xvYWQvc2NoZW1hY2hhbmdlOnNjaGVtYWNoYW5nZV90ZXN0) on master @ [386b3da7f0d31a43ca66709fdaa30326fd0624b1](https://github.com/cockroachdb/cockroach/commits/386b3da7f0d31a43ca66709fdaa30326fd0624b1):


```
    "result": false
   },
   {
    "query": "SELECT count(*) FROM public.table_w1_9",
    "result": 1
   }
  ],
  "previousStatements": [
   "ALTER TABLE public.table_w1_9 ADD CONSTRAINT table_w1_58_col58_w1_63_table_w1_9_col9_w1_17_fk FOREIGN KEY (col9_w1_17) REFERENCES public.table_w1_58 (col58_w1_63) ON DELETE CASCADE ON UPDATE CASCADE"
  ],
  "usesDeclarativeSchemaChanger": true
 }
}
    schema_change_external_test.go:134: 
        	Error Trace:	pkg/ccl/testccl/workload/schemachange/schemachange_test_test/pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:134
        	Error:      	Received unexpected error:
        	            	***UNEXPECTED ERROR; Received an unexpected execution error.: ERROR: current transaction is aborted, commands ignored until end of transaction block (SQLSTATE 25P02)
        	            	(1) forced error mark
        	            	  | "fatal error when running txn"
        	            	  | github.com/cockroachdb/errors/withstack/*withstack.withStack::
        	            	Wraps: (2)
        	            	Wraps: (3) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*opStmt).executeStmt
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/operation_generator.go:3178
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).runInTxn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/schemachange.go:499
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).run
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/schemachange.go:587
        	            	  | pkg/ccl/testccl/workload/schemachange/schemachange_test_test.TestWorkload.TestWorkload.func4.func6
        	            	  | 	pkg/ccl/testccl/workload/schemachange/schemachange_test_test/pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:123
        	            	  | golang.org/x/sync/errgroup.(*Group).Go.func1
        	            	  | 	golang.org/x/sync/errgroup/external/org_golang_x_sync/errgroup/errgroup.go:78
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1695
        	            	Wraps: (4) ***UNEXPECTED ERROR; Received an unexpected execution error.
        	            	Wraps: (5) ERROR: current transaction is aborted, commands ignored until end of transaction block (SQLSTATE 25P02)
        	            	Error types: (1) *markers.withMark (2) *schemachange.ErrorState (3) *withstack.withStack (4) *errutil.withPrefix (5) *pgconn.PgError
        	Test:       	TestWorkload
Schema Workload Stats
Total Schema Statements Executed = 63
Total Schema Statements Succeeded = 63
Total Schema Statement Expected Failures = 0
Total Transactions Committed = 42
Total Transactions Rolled Back = 30
Total Transactions Executed = 72
    schema_change_external_test.go:99: backup, tracing data, and system table dumps in /var/lib/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/testlogs/pkg/ccl/testccl/workload/schemachange/schemachange_test/run_21_of_25/test.outputs/logTestWorkload2252033904
    panic.go:626: -- test log scope end --
test logs left over in: outputs.zip/logTestWorkload2252033904
--- FAIL: TestWorkload (18.53s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=21</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #131002 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot T-sql-foundations branch-release-24.1.5-rc]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestWorkload.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

pkg/ccl/testccl/workload/schemachange/schemachange_test.TestWorkload [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17160903?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17160903?buildTab=artifacts#/) on master @ [2b80ff9855cc2974e046d41b760a00adbce6e6fb](https://github.com/cockroachdb/cockroach/commits/2b80ff9855cc2974e046d41b760a00adbce6e6fb):


```
    "queryArgs": [
     "udf_w1_71",
     ""
    ],
    "result": false
   }
  ],
  "previousStatements": [
   "CREATE FUNCTION schema_w1_2.udf_w1_71()\n\tRETURNS VARCHAR\n\tLANGUAGE SQL\n\tAS $$ SELECT NULL WHERE ((SELECT count(*) FROM public.table_w1_3 LIMIT 0) = 0) AND ((SELECT count(*) FROM schema_w1_2.table_w0_173 LIMIT 0) = 0) AND ((SELECT count(*) FROM schema_w1_2.table_w0_233 LIMIT 0) = 0) AND ((SELECT count(*) FROM public.table_w0_10 LIMIT 0) = 0) AND ((SELECT count(*) FROM public.table_w0_180 LIMIT 0) = 0) AND ((SELECT count(*) FROM public.table_w0_200 LIMIT 0) = 0) AND (SELECT public.udf_w1_25(e'\\x1d':::STRING,B'0000111110110111110110001011100000110101011010001100001100111010011010111000010000111101111111011',true,'BOX(-0.3733134660469342 -0.6724445219040603,-0.23999742209819228 1.3540914437296516)':::BOX2D,'R':::STRING,'\\x82f05adf4c':::BYTES,e'e\\x02\\x07gCn\\tc3':::STRING,e'\\'':::STRING,B'001100110110100101011001011010011100000100011010001101100010000101',NULL) IS NOT NULL) $$"
  ],
  "usesDeclarativeSchemaChanger": true
 }
}
    schema_change_external_test.go:131: 
        	Error Trace:	pkg/ccl/testccl/workload/schemachange/schemachange_test_test/pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:131
        	Error:      	Received unexpected error:
        	            	***UNEXPECTED ERROR; Received an unexpected execution error.: ERROR: return type mismatch in function declared to return table_w1_3 (SQLSTATE 42P13)
        	            	(1) forced error mark
        	            	  | "fatal error when running txn"
        	            	  | github.com/cockroachdb/errors/withstack/*withstack.withStack::
        	            	Wraps: (2)
        	            	Wraps: (3) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*opStmt).executeStmt
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/operation_generator.go:3173
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).runInTxn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/schemachange.go:494
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).run
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/schemachange.go:582
        	            	  | pkg/ccl/testccl/workload/schemachange/schemachange_test_test.TestWorkload.TestWorkload.func4.func6
        	            	  | 	pkg/ccl/testccl/workload/schemachange/schemachange_test_test/pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:120
        	            	  | golang.org/x/sync/errgroup.(*Group).Go.func1
        	            	  | 	golang.org/x/sync/errgroup/external/org_golang_x_sync/errgroup/errgroup.go:78
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_arm64.s:1222
        	            	Wraps: (4) ***UNEXPECTED ERROR; Received an unexpected execution error.
        	            	Wraps: (5) ERROR: return type mismatch in function declared to return table_w1_3 (SQLSTATE 42P13)
        	            	Error types: (1) *markers.withMark (2) *schemachange.ErrorState (3) *withstack.withStack (4) *errutil.withPrefix (5) *pgconn.PgError
        	Test:       	TestWorkload
Schema Workload Stats
Total Schema Statements Executed = 67
Total Schema Statements Succeeded = 67
Total Schema Statement Expected Failures = 0
Total Transactions Committed = 36
Total Transactions Rolled Back = 33
Total Transactions Executed = 69
    schema_change_external_test.go:96: backup, tracing data, and system table dumps in /artifacts/tmp/_tmp/67537770b84c3a4ea177539c1e46bd43/logTestWorkload3762702916
    panic.go:626: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/67537770b84c3a4ea177539c1e46bd43/logTestWorkload3762702916
--- FAIL: TestWorkload (9.73s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestWorkload.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

--- a/pkg/workload/schemachange/error_screening.go
+++ b/pkg/workload/schemachange/error_screening.go
@@ -927,7 +927,7 @@ func (og *operationGenerator) columnHasSingleUniqueConstraint(
 	           AND table_name = $2
 	           AND column_name = $3
 	           AND (contype = 'u' OR contype = 'p')
-	           AND array_length(conkey, 1) = 1
+	           AND array_length(conkey, 1) >= 1 -- unique index has to have this a prefix
 					   AND conkey[1] = ordinal_position
 	       )
 	`, tableName.Schema(), tableName.Object(), columnName)
@@ -1030,15 +1030,20 @@ SELECT count(*) FROM %s
 			WHERE t2.%s IS NOT NULL
 `, childTable.String(), parentTable.String(), childColumn.name.String(), parentColumn.name.String(), parentColumn.name.String())
 
-	numJoinRows, err := og.scanInt(ctx, tx, q)
+	joinTx, err := tx.Begin(ctx)
 	if err != nil {
-		// UndefinedFunction errors mean that the column type is not comparable.
-		if pgErr := new(pgconn.PgError); errors.As(err, &pgErr) && pgcode.MakeCode(pgErr.Code) == pgcode.UndefinedFunction {
-			return false, nil
-		}
 		return false, err
 	}
-	return numJoinRows == childRows, err
+	numJoinRows, err := og.scanInt(ctx, joinTx, q)
+	if err != nil {
+		rbkErr := joinTx.Rollback(ctx)
+		// UndefinedFunction errors mean that the column type is not comparable.
+		if pgErr := new(pgconn.PgError); errors.As(err, &pgErr) && pgcode.MakeCode(pgErr.Code) == pgcode.UndefinedFunction {
+			return false, rbkErr
+		}
+		return false, errors.WithSecondaryError(err, rbkErr)
+	}
+	return numJoinRows == childRows, joinTx.Commit(ctx)
 }
 
 var (
@@ -1218,8 +1223,9 @@ func (og *operationGenerator) violatesFkConstraintsHelper(
 	}
 	checkSharedParentChildRows := ""
 	if len(parentAndChildSameQueryColumns) > 0 {
-		// Ensure none of the rows being inserted satisfy the constraint, since
-		//// its referring to itself.
+		// Check if none of the rows being inserted satisfy the foreign key constraint,
+		// since the foreign key constraints refers to the same table. So, anything in
+		// the insert batch can satisfy the constraint.
 		checkSharedParentChildRows = fmt.Sprintf("NOT (true = ANY (ARRAY [%s])) AND",
 			strings.Join(parentAndChildSameQueryColumns, ","))
 	}

--- a/pkg/workload/schemachange/error_screening.go
+++ b/pkg/workload/schemachange/error_screening.go
@@ -16,6 +16,7 @@ import (
 	"regexp"
 	"strings"
 
+	"github.com/cockroachdb/cockroach/pkg/sql/parser"
 	"github.com/cockroachdb/cockroach/pkg/sql/pgwire/pgcode"
 	"github.com/cockroachdb/cockroach/pkg/sql/sem/tree"
 	"github.com/cockroachdb/cockroach/pkg/sql/types"
@@ -337,20 +338,62 @@ func (og *operationGenerator) columnIsDependedOn(
 func (og *operationGenerator) colIsRefByComputed(
 	ctx context.Context, tx pgx.Tx, tableName *tree.TableName, columnName tree.Name,
 ) (bool, error) {
-	return og.scanBool(ctx, tx, `SELECT EXISTS(
-    SELECT
-       attrelid::REGCLASS AS table_name,
-       attname AS column_name,
-       pg_get_expr(adbin, adrelid) AS computed_formula
-    FROM
-       pg_attribute
-    JOIN
-       pg_attrdef ON attrelid = adrelid AND attnum = adnum
-    WHERE
-       atthasdef
-       AND attrelid = $1::REGCLASS
-       AND pg_get_expr(adbin, adrelid) ILIKE '%%' || $2 || '%%'
-)`, tableName.String(), columnName)
+	colIsRefByGeneratedExpr := false
+
+	query := `WITH tab_json AS (
+		SELECT crdb_internal.pb_to_json(
+		'desc',
+		descriptor
+	)->'table' AS t
+	FROM system.descriptor
+	WHERE id = $1::REGCLASS
+	),
+	columns_json AS (
+		SELECT json_array_elements(t->'columns') AS c FROM tab_json
+	),
+	columns AS (
+		SELECT c->>'computeExpr' AS generation_expression,
+		c->>'name' AS column_name,
+		c->>'id' AS ordinal
+	FROM columns_json
+	)
+	SELECT generation_expression FROM columns WHERE generation_expression IS NOT NULL
+	`
+	generatedExpressions, err := Collect[string](ctx, og, tx, pgx.RowTo[string], query, tableName.String())
+	if err != nil {
+		return false, err
+	}
+	for _, generatedExpression := range generatedExpressions {
+		expr, err := parser.ParseExpr(generatedExpression)
+		if err != nil {
+			return false, err
+		}
+
+		if _, err := tree.SimpleVisit(expr, func(expr tree.Expr) (recurse bool, newExpr tree.Expr, err error) {
+			vBase, ok := expr.(tree.VarName)
+			if !ok {
+				return true, expr, nil
+			}
+
+			v, err := vBase.NormalizeVarName()
+			if err != nil {
+				return false, nil, err
+			}
+
+			c, ok := v.(*tree.ColumnItem)
+			if !ok {
+				return true, expr, nil
+			}
+			if c.ColumnName == columnName {
+				colIsRefByGeneratedExpr = true
+				return false, expr, nil
+			}
+			return true, expr, nil
+		}); err != nil {
+			return false, err
+		}
+	}
+	return colIsRefByGeneratedExpr, nil
 }
 
 func (og *operationGenerator) columnIsDependedOnByView(

--- a/pkg/workload/schemachange/error_screening.go
+++ b/pkg/workload/schemachange/error_screening.go
@@ -1218,7 +1218,9 @@ func (og *operationGenerator) violatesFkConstraintsHelper(
 	}
 	checkSharedParentChildRows := ""
 	if len(parentAndChildSameQueryColumns) > 0 {
-		checkSharedParentChildRows = fmt.Sprintf("false = ANY (ARRAY [%s]) AND",
+		// Ensure none of the rows being inserted satisfy the constraint, since
+		//// its referring to itself.
+		checkSharedParentChildRows = fmt.Sprintf("NOT (true = ANY (ARRAY [%s])) AND",
 			strings.Join(parentAndChildSameQueryColumns, ","))
 	}
 	q := fmt.Sprintf(`

--- a/pkg/workload/schemachange/operation_generator.go
+++ b/pkg/workload/schemachange/operation_generator.go
@@ -323,6 +323,16 @@ func (og *operationGenerator) addColumn(ctx context.Context, tx pgx.Tx) (*opStmt
 			code: pgcode.FeatureNotSupported, condition: isJSONTyp,
 		},
 	})
+	// There is a risk of unique violations if concurrent inserts
+	// happen during an ADD COLUMN UNIQUE / NOT NULL. So allow this to be
+	// a potential error on the commit.
+	if def.Unique.IsUnique {
+		og.potentialCommitErrors.add(pgcode.UniqueViolation)
+	}
+	if def.Nullable.Nullability == tree.NotNull {
+		og.potentialCommitErrors.add(pgcode.NotNullViolation)
+	}
+
 	op.sql = fmt.Sprintf(`ALTER TABLE %s ADD COLUMN %s`, tableName, tree.AsString(def))
 	return op, nil
 }
@@ -2445,6 +2455,8 @@ func (og *operationGenerator) setColumnNotNull(ctx context.Context, tx pgx.Tx) (
 		}
 		if colContainsNull {
 			og.candidateExpectedCommitErrors.add(pgcode.NotNullViolation)
+			// If executed within the same txn as CREATE TABLE.
+			stmt.potentialExecErrors.add(pgcode.NotNullViolation)
 		}
 	}
 
@@ -2754,6 +2766,11 @@ func (og *operationGenerator) alterTableAlterPrimaryKey(
 	// TODO(sql-foundations): Until #130165 is resolved, we add this potential
 	// error.
 	og.potentialCommitErrors.add(pgcode.DuplicateColumn)
+	// There is a risk of unique violations if concurrent inserts
+	// happen during an ALTER PRIMARY KEY. So allow this to be
+	// a potential error on the commit.
+	og.potentialCommitErrors.add(pgcode.UniqueViolation)
+
 	opStmt := newOpStmt(stmt, codesWithConditions{
 		{code, true},
 	})

--- a/pkg/workload/schemachange/operation_generator.go
+++ b/pkg/workload/schemachange/operation_generator.go
@@ -1660,11 +1660,6 @@ func (og *operationGenerator) dropColumn(ctx context.Context, tx pgx.Tx) (*opStm
 		{code: pgcode.DependentObjectsStillExist, condition: columnIsDependedOn || columnRemovalWillDropFKBackingIndexes},
 		{code: pgcode.FeatureNotSupported, condition: hasAlterPKSchemaChange},
 	})
-	// TODO(#126967): We need to add a check for the column being in an expression
-	// to an index. In the case where the expression does not already exist for
-	// us to use, we add an internal crdb_internal_idx_expr prefixed column to
-	// the table.
-	stmt.potentialExecErrors.add(pgcode.InvalidColumnReference)
 	// For legacy schema changer its possible for create index operations to interfere if they
 	// are in progress.
 	if !og.useDeclarativeSchemaChanger {

--- a/pkg/workload/schemachange/schemachange.go
+++ b/pkg/workload/schemachange/schemachange.go
@@ -20,6 +20,7 @@ import (
 	"io"
 	"os"
 	"regexp"
+	"strings"
 	"sync"
 	"sync/atomic"
 	"time"
@@ -508,6 +509,15 @@ func (w *schemaChangeWorker) runInTxn(
 				// to rollback.
 				if pgcode.MakeCode(pgErr.Code) == pgcode.SerializationFailure {
 					w.recordInHist(timeutil.Since(start), txnRollback)
+					return errors.Mark(
+						err,
+						errRunInTxnRbkSentinel,
+					)
+				}
+				// Command is too large errors are allowed on DML operations since,
+				// some of the tables can be pretty wide in this test.
+				if op.queryType == OpStmtDML && pgcode.MakeCode(pgErr.Code) == pgcode.Uncategorized &&
+					strings.Contains(pgErr.Error(), "command is too large") {
 					return errors.Mark(
 						err,
 						errRunInTxnRbkSentinel,

